--- old
+++ new
@@ -57,3 +57,3 @@
-Penner rumlaufen will. Ein bisschen Damenzeug, Zahnbürste, ein Stück Seife. Oben
-drauf der wichtigste Schatz: mein Matheblock. Plus ein paar Stifte, damit ich
-nicht wie eine Idiotin dasitze und nix schreiben kann.
+Penner rumlaufen will. Ein paar Binden, Zahnbürste, ein Stück Seife. Oben drauf
+der wichtigste Schatz: mein Matheblock. Plus ein paar Stifte, damit ich nicht
+wie eine Idiotin dasitze und nix schreiben kann.
@@ -68,1 +68,1 @@
-Weiter jetzt. Keine Zeit für Jammern.
+Weiter jetzt. Keine Zeit zum Jammern.