--- old
+++ new
@@ -43,1 +43,1 @@
-Nochmal Einschlafen kann ich vergessen. Ich hab mich jetzt so lange hin- und
+Noch mal einschlafen kann ich vergessen. Ich hab mich jetzt so lange hin- und
@@ -61,2 +61,2 @@
-sein? Haha. Als ob das geht. Irgendwie absurd: Ein bisschen Stoff, ein paar
-Zettel, und damit will ich durchkommen?
+sein? Haha. Als ob das geht. Irgendwie absurd: ein bisschen Stoff, ein paar
+Zettel – und damit will ich durchkommen?
@@ -73,1 +73,1 @@
-überallhin begleitet, sogar heimlich in Ferienlager, wo’s eigentlich schon
+überallhin begleitet, sogar heimlich ins Ferienlager, wo’s eigentlich schon
@@ -80,1 +80,1 @@
-Ich streich einmal mit den Fingern über den Stoff, dann schieb ich die Tasche
+Ich streiche einmal mit den Fingern über den Stoff, dann schiebe ich die Tasche
@@ -88,1 +88,0 @@
-