--- old
+++ new
@@ -9,2 +9,4 @@
-  "persons": ["Eddie"],
-  "event": "Schlaflos beschließt Eddie, ihre Fluchtvorbereitungen ernsthaft zu beginnen. Im Licht der Mittsommersonne packt sie ihre kleine Reisetasche mit Unterwäsche, Trainingshose, Hygieneartikeln, Seife, Zahnbürste, ihrem Matheblock und Stiften. Sie erkennt, dass sie ihre große Tasche, viele Klamotten, ihr Kuscheltier und symbolisch auch die DDR und ihre Kindheit zurücklassen muss. Ein kurzer Moment des Zweifelns, doch sie wischt ihn beiseite. Bald wird der Bus nach Joutsa zum zweiten Wettkampftag fahren.",
+  "persons": [
+    "Eddie"
+  ],
+  "synopsis": "Schlaflos beschließt Eddie, ihre Fluchtvorbereitungen ernsthaft zu beginnen. Im Licht der Mittsommersonne packt sie ihre kleine Reisetasche mit Unterwäsche, Trainingshose, Hygieneartikeln, Seife, Zahnbürste, ihrem Matheblock und Stiften. Sie erkennt, dass sie ihre große Tasche, viele Klamotten, ihr Kuscheltier und symbolisch auch die DDR und ihre Kindheit zurücklassen muss. Ein kurzer Moment des Zweifelns, doch sie wischt ihn beiseite. Bald wird der Bus nach Joutsa zum zweiten Wettkampftag fahren.",
@@ -14,1 +16,2 @@
-}```
+}
+```