--- old
+++ new
@@ -53,2 +53,2 @@
-reicht: extrem unwahrscheinlich. Mir sind nur noch 15 Kronen von Aarons
-Geld geblieben. Ich müsste meine D-Mark in der Bank in Kronen tauschen.
+reicht: extrem unwahrscheinlich. Mir sind nur noch 15 Markka von Aarons
+Geld geblieben. Ich müsste meine D-Mark in der Bank in Markka tauschen.
@@ -59,1 +59,1 @@
-sein und versuchen mit meinen 15 Kronen nach Schweden zu kommen.
+sein und versuchen mit meinen 15 Markka nach Schweden zu kommen.