--- old
+++ new
@@ -56,1 +56,1 @@
-Streiten sie wegen mir? Bestimmt.
+Streiten sie meinetwegen? Bestimmt.
@@ -62,3 +62,3 @@
-um, ihr Blick ist weich, gar nicht wie eben die Stimmen. „Hyvää huomenta,
-Eddie.“ Ihre Stimme klingt, als hätte es nie Streit gegeben. Auf dem Tisch
-wartet Frühstück – Brot, Käse, Gurken, sogar eine Tasse Kakao. Für mich.
+um, ihr Blick ist weich, gar nicht wie eben die Stimmen. „Hyvää huomenta, Eddie.
+Guten Morgen!“ Ihre Stimme klingt, als hätte es nie Streit gegeben. Auf dem
+Tisch wartet Frühstück – Brot, Käse, Gurken, sogar eine Tasse Kakao. Für mich.