--- old
+++ new
@@ -106,3 +106,3 @@
-sie sofort merken, dass meine Wuppertal Legende nicht stimmt. Und was, wenn es…
-Ostdeutsche sind? Nein, das wäre fast unmöglich, oder? Aber ich traue mich
-nicht, nähere mich ihnen vorsichtig und halte einigem Abstand, bis ich die
+sie sofort merken, dass meine Wuppertal-Legende nicht stimmt. Und was, wenn es…
+Ostdeutsche sind? Nein, das wäre fast unmöglich, oder? Aber ich traue mich nicht
+so recht, nähere mich ihnen vorsichtig und halte einigen Abstand, bis ich die
@@ -134,1 +134,1 @@
-Pouch-Faltboote aus der DDR. Wie können hier mitten in Finnland vier Ostdeutsche
+Pouch-Faltboote aus der DDR. Wie können hier mitten in Finnland vier ostdeutsche
@@ -147,1 +147,1 @@
-Mädchen zu. Blond, lachend, ein offender Blick – als ob ich längst zur Gruppe
+Mädchen zu. Blond, lachend, ein offener Blick – als ob ich längst zur Gruppe
@@ -156,1 +156,1 @@
-„Eddie“ stammele ich, doch bevor ich weiter antworten kann, fällt mir Conny
+„Eddie“, stammele ich, doch bevor ich weiter antworten kann, fällt mir Conny
@@ -196,1 +196,1 @@
-„Also Eddie“, sagt Sabine schließlich, „jetzt sind wir mal gespannt. Wo kommst
+„Also, Eddie“, sagt Sabine schließlich, „jetzt sind wir mal gespannt. Wo kommst