--- old
+++ new
@@ -68,1 +68,1 @@
-„Avaisit minun postini?!“ – „Du öffnest meine Post?!“ Sini klingt fassungslos,
+„Avasit minun postini?!“ – „Du öffnest meine Post?!“ Sini klingt fassungslos,
@@ -94,1 +94,1 @@
-dem Wohnheim gesprochen, ein Zimmer für dich reserviert.“
+dem Wohnheim gesprochen und ein Zimmer für dich reserviert.“
@@ -124,2 +124,2 @@
-„Kiitos, Danke Eddie…“, heult Sini. „Ich wollte auch nicht, dass es je endet.
-Aber…“ setzt sie verlegen an. „…ich wußte es schon ganz lange.“. „Mit der
+„Kiitos, Danke Eddie…“ heult Sini. „Ich wollte auch nicht, dass es je endet.
+Aber…“ setzt sie verlegen an. „…ich wusste es schon ganz lange.“ „Mit der
@@ -128,1 +128,1 @@
-verloren.“ Ich verstehe. „Aber wie hast Du es letzten Winter gemacht?“ - „Da
+verloren.“ Ich verstehe. „Aber wie hast Du es letzten Winter gemacht?“ – „Da
@@ -131,4 +131,4 @@
-Wir sitzen still nebeneinander, die Tränen fast versiegt, nur unsere Atemzüge
-füllen das kleine Zimmer. Draußen rauscht der Wind durch die dunklen Bäume, aber
-hier drinnen halten wir uns fest wie zwei, die noch längst nicht aufgeben
-wollen.
+Wir sitzen still nebeneinander, die Tränen sind fast versiegt, nur unsere
+Atemzüge füllen das kleine Zimmer. Draußen rauscht der Wind durch die dunklen
+Bäume, aber hier drinnen halten wir uns fest wie zwei, die noch längst nicht
+aufgeben wollen.