--- old
+++ new
@@ -56,3 +56,2 @@
-voller Schätze. Ich zog einen Schuber heraus, und da waren sie:
-Messetischblätter von Finnland, fein säuberlich aufgereiht. Mein Herz schlug bis
-zum Hals.
+voller Schätze. Ich zog einen Schuber heraus, und da waren sie: Messtischblätter
+von Finnland, fein säuberlich aufgereiht. Mein Herz schlug bis zum Hals.
@@ -70,1 +69,1 @@
-Zuhause nahm mein „Fluchtblock“ Gestalt an. Nach außen ein stinknormaler
+Zu Hause nahm mein „Fluchtblock“ Gestalt an. Nach außen ein stinknormaler
@@ -73,1 +72,1 @@
-gewesen. Also verwandelte ich sie in Geometrie. So bildeted drei wichtige Städte
+gewesen. Also verwandelte ich sie in Geometrie. So bildeten drei wichtige Städte
@@ -79,3 +78,3 @@
-saßen wir zusammen am Küchentisch, beugten uns über Linien und Zahlen, und er
-tat so, als erkläre er mir nur eine knifflige Matheaufgabe. In Wahrheit aber
-legten wir die Landkarte für meine Zukunft.
+saßen wir zusammen am Küchentisch, beugten uns über Linien und Zahlen und er tat
+so, als erkläre er mir nur eine knifflige Matheaufgabe. In Wahrheit aber legten
+wir die Landkarte für meine Zukunft.