--- old
+++ new
@@ -7,0 +7,1 @@
+  "day": "Friday",
@@ -59,4 +60,5 @@
-Ich wusste, dass ich sie unmöglich mitnehmen konnte. Also schmuggelte ich sie in
-die öffentlich zugängliche Kartensammlung, versteckt unter den mongolischen
-Karten, die sich eh keiner anschaute. Von da an hatte ich *meine Karten*. Fast
-jeden Tag saß ich in diesem muffigen Raum und studierte Finnland.
+Ich wusste, dass ich sie unmöglich mit nach Hause nehmen konnte. Also
+schmuggelte ich sie in die öffentlich zugängliche Kartensammlung, versteckt
+unter den mongolischen Karten, die sich eh keiner anschaute. Von da an hatte ich
+*meine Karten*. Fast jeden Tag saß ich in diesem muffigen Raum und studierte
+Finnland.