--- old
+++ new
@@ -20,15 +20,2 @@
-  "feedback": {
-    "anni": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "lucia": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "soek": {
-      "rating": ".....",
-      "comments": ""
-    }
-  },
-  "todo": "dieser und der nächste Abschnitt passen nicht zusammen"
+  "todo": "dieser und der nächste Abschnitt passen nicht zusammen",
+  "proofreading": {}
@@ -96,2 +83,2 @@
-Trockner, Regalen voller Waschmittel. Ich schlucke. „Du kannst dir Zeit lassen",
-sagt er sanft, „ich lass dich allein." Er verschwindet, und ich bleibe zurück,
+Trockner, Regalen voller Waschmittel. Ich schlucke. „Du kannst dir Zeit lassen“,
+sagt er sanft, „ich lass dich allein.“ Er verschwindet, und ich bleibe zurück,
@@ -111,1 +98,1 @@
-Augen weiten sich, er stößt ein hastiges „Oh!" aus, reißt den Blick weg und
+Augen weiten sich, er stößt ein hastiges „Oh!“ aus, reißt den Blick weg und