--- old
+++ new
@@ -15,1 +15,1 @@
-  Rückblickend ließe sich vieles einordnen, kommentieren, erklären. Aber das würde die Intensität der existenziellen Situation schwächen. Eddie weiß im Moment der Handlung nicht, ob sie überlebt, und die Lesenden sollen diesen Schwebezustand teilen.
+  Rückblickend ließe sich vieles einordnen, kommentieren, erklären. Aber das würde die Intensität der existenziellen Situation schwächen. Eddie weiß im Moment der Handlung nicht, ob ihre Flucht gelingt, sie ins Gefängnis oder in die Freiheit kommt. Die Lesenden sollen diesen Schwebezustand teilen.