--- old
+++ new
@@ -20,14 +20,1 @@
-  "feedback": {
-    "anni": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "lucia": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "soek": {
-      "rating": ".....",
-      "comments": ""
-    }
-  }
+  "proofreading": {}
@@ -53,1 +40,1 @@
-nun tatsächlich statt!"
+nun tatsächlich statt!“
@@ -62,1 +49,1 @@
-nimmst Du Dir ja sehr zu Herzen, mein Lieber…"
+nimmst Du Dir ja sehr zu Herzen, mein Lieber…“