--- old
+++ new
@@ -56,1 +56,1 @@
-diese Handbewegung darfst Du nie machen! Das ist streng verboten. Geht gar
+diese Handbewegung darfst du nie machen! Das ist streng verboten. Geht gar
@@ -62,2 +62,2 @@
-Ich lache noch immer Tränen. „Nein! Ganz falsches Land, ganz falsche Zeit, ganz
-falsche Geste!“
+Ich lache so sehr, dass mir Tränen kommen. „Nein! Ganz falsches Land, ganz
+falsche Zeit, ganz falsche Geste!“