--- old
+++ new
@@ -126,2 +126,2 @@
-bist besser drauf. Heute erlaube ich Dir den ersten kleinen Spaziergang.
-Erinnerst Du Dich an die Bänke am Hafen“
+bist besser drauf. Heute erlaube ich dir den ersten kleinen Spaziergang.
+Erinnerst du dich an die Bänke am Hafen?“
@@ -131,3 +131,3 @@
-„Sei so lieb und warte da so gegen halb sieben auf mich - Aber Kein Schritt zum
-Terminal, verstanden? Das ist viel zu weit!“ Sinis plötzliche Strenge ist nicht
-nur gespielt. Sie meint es ernst.
+„Sei so lieb und warte da so gegen halb sieben auf mich – aber keinen Schritt
+zum Terminal, verstanden? Das ist viel zu weit!“ Sinis plötzliche Strenge ist
+nicht nur gespielt. Sie meint es ernst.
@@ -135,1 +135,1 @@
-„Ja Frau General!“ - Sini grinst und ist auch schon aus der Tür.
+„Ja, Frau General!“ – Sini grinst und ist auch schon aus der Tür.
@@ -164,1 +164,1 @@
-Ein Regal steht voller finnischer und schwedischer Biologielehrbücher. Momemt,
+Ein Regal steht voller finnischer und schwedischer Biologielehrbücher. Moment,
@@ -166,1 +166,1 @@
-Ich schlage es auf, blättere, und bald fange ich an zu kichern. Eine Idee nimmt
+Ich schlage es auf, blättere und bald fange ich an zu kichern. Eine Idee nimmt
@@ -171,1 +171,1 @@
-dem Bad kommt. Der andere Strang ist artiger: ich sticke ihm bunte Sámi-Muster
+dem Bad kommt. Der andere Strang ist artiger: Ich sticke ihm bunte Sámi-Muster
@@ -174,1 +174,1 @@
-Die Codons sind nicht zufällig: ich suche im Buch nach den Sequenzen, die für
+Die Codons sind nicht zufällig: Ich suche im Buch nach den Sequenzen, die für
@@ -192,5 +192,5 @@
-Sini hat mir den Weg beschrieben, und tatsächlich: es ist leicht. Ich laufe
-langsam, nehme Rücksicht auf meine Füße. Erinnere mich an Sinis „keine Eile“.
-Die frische Abendluft tut mir gut. Laufen weil man möchte, nicht weil man muss.
-Als ich bei den Bänken ankomme, sehe ich schon ganz klein Sini über die Brücke
-laufen. Ihr blauer Iro wippt unverkennbar.
+Sini hat mir den Weg beschrieben, und tatsächlich: Es ist leicht. Ich laufe
+langsam, nehme Rücksicht auf meine Füße. Ich erinnere mich an Sinis „keine
+Eile“. Die frische Abendluft tut mir gut. Laufen, weil man möchte, nicht weil
+man muss. Als ich bei den Bänken ankomme, sehe ich Sini schon ganz klein über
+die Brücke laufen. Ihr blauer Iro wippt unverkennbar.
@@ -198,1 +198,1 @@
-Ich winke und setze mich auf schönste Holzbank am kleinen Stadthafen, genau
+Ich winke und setze mich auf die schönste Holzbank am kleinen Stadthafen, genau
@@ -204,2 +204,2 @@
-„Wie geht es dir?“ frage ich schnell zuerst. Sie lacht: „Joo, ihan jees.“ - „Ja,
-ganz okay. Und wie geht es Dir?“
+„Wie geht es dir?“ frage ich schnell zuerst. Sie lacht: „Joo, ihan jees.“ – „Ja,
+ganz okay. Und wie geht es dir?“
@@ -207,2 +207,2 @@
-„Ich lebe Sini. Ich lebe. Ist das nicht wunderbar?“ - „Joo, ihan jees.“ erwidert
-sie lachend.
+„Ich lebe, Sini. Ich lebe. Ist das nicht wunderbar?“ – „Joo, ihan jees“,
+erwidert sie lachend.
@@ -213,3 +213,3 @@
-„Heute kümmere ich mich ums Abendessen.“ Sie zieht mich weiter, in eine
-Pizzeria. Drinnen sind die Fenster beschlagen, der Boden klebt, die Luft riecht
-nach Oregano.
+„Heute kümmere ich mich ums Abendessen.“ Sie zieht mich weiter in eine Pizzeria.
+Drinnen sind die Fenster beschlagen, der Boden klebt, die Luft riecht nach
+Oregano.