--- old
+++ new
@@ -17,15 +17,2 @@
-  "feedback": {
-    "anni": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "lucia": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "soek": {
-      "rating": ".....",
-      "comments": ""
-    }
-  },
-  "todo": "dieser und der vorherige Abschnitt passen nicht zusammen"
+  "todo": "dieser und der vorherige Abschnitt passen nicht zusammen",
+  "proofreading": {}
@@ -75,1 +62,1 @@
-diesen treuen Augen an. „Wir müssen ihn finden", flüstere ich. Meine Stimme
+diesen treuen Augen an. „Wir müssen ihn finden“, flüstere ich. Meine Stimme
@@ -77,1 +64,1 @@
-einzige."
+einzige.“