--- old
+++ new
@@ -71,1 +71,1 @@
-würde ich die Sachen zufällig finden. Aber jetzt... jetzt weiß ich, was es
+würde ich die Sachen zufällig finden. Aber jetzt… jetzt weiß ich, was es
@@ -91,4 +91,4 @@
-„Kennst du... kennst du vielleicht die Stadtmission? Ich… ich müsste hin. Aber
-ich traue mich nicht allein. Kannst du… vielleicht vorgehen? Nur schauen, ob
-jemand da ist. Polizei oder so.“ Meine Stimme bricht fast. Ich erwarte, dass er
-lacht oder sich einfach umdreht.
+„Kennst du… kennst du vielleicht die Stadtmission? Ich… ich müsste hin. Aber ich
+traue mich nicht allein. Kannst du… vielleicht vorgehen? Nur schauen, ob jemand
+da ist. Polizei oder so.“ Meine Stimme bricht fast. Ich erwarte, dass er lacht
+oder sich einfach umdreht.