--- old
+++ new
@@ -54,1 +54,1 @@
-„Bonjour, hier spricht das Sekretariat des Institut de..." -- ich werfe Raija
+„Bonjour, hier spricht das Sekretariat des Institut de…" -- ich werfe Raija
@@ -59,3 +59,2 @@
-„...des *Institut de Sciences Avancées*", wiederhole ich schnell. „Ich rufe an,
-um Ihnen mitzuteilen, dass Sie für den Prix de sûreté nucléaire nominiert
-wurden."
+„…des *Institut de Sciences Avancées*", wiederhole ich schnell. „Ich rufe an, um
+Ihnen mitzuteilen, dass Sie für den Prix de sûreté nucléaire nominiert wurden."
@@ -64,2 +63,2 @@
-gedehnt, und dann viel lauter: „Oh! Na, das ist ja... das ist ja wirklich...
-also, das ist großartig! Aus Frankreich, sagen Sie?"
+gedehnt, und dann viel lauter: „Oh! Na, das ist ja… das ist ja wirklich… also,
+das ist großartig! Aus Frankreich, sagen Sie?"
@@ -127,1 +126,1 @@
-Fluchtkünstlerin. Und jetzt... trinken wir einen Kaffee. Du siehst aus, als
+Fluchtkünstlerin. Und jetzt… trinken wir einen Kaffee. Du siehst aus, als