--- old
+++ new
@@ -64,1 +64,1 @@
-„Bonjour, hier spricht das Sekretariat des Institut de…“ -- ich werfe Raija
+„Bonjour, hier spricht das Sekretariat des Instituts de…“ – ich werfe Raija
@@ -84,4 +84,4 @@
-„Ja, natürlich, natürlich!“ tönt er. „Das muss ich sofort meiner Frau erzählen
--- und den Kollegen! Wissen Sie, ich habe ja schon lange darauf gewartet, dass
-Paris meine Arbeit würdigt.“ um dann noch auf breitem sächsisch ein
-„formidable!“ hintendran zu hängen.
+„Ja, natürlich, natürlich!“ tönt er. „Das muss ich sofort meiner Frau erzählen –
+und den Kollegen! Wissen Sie, ich habe ja schon lange darauf gewartet, dass
+Paris meine Arbeit würdigt“, setzt er nach und hängt dann auf breitem Sächsisch
+noch ein „formidable!“ hinten dran.
@@ -94,1 +94,1 @@
-„Bonne journée“, murmele ich schnell, und lege auf, bevor mir doch noch die
+„Bonne journée“, murmele ich schnell und lege auf, bevor mir doch noch die
@@ -108,2 +108,2 @@
-überwacht, wenn nicht auch schon davor. Jemand anderen wollten wir in die Sache
-nicht mit hineinziehen.
+überwacht, wenn nicht auch schon davor. Jemanden anderen wollten wir in die
+Sache nicht mit hineinziehen.
@@ -111,1 +111,1 @@
-Jemand anderen? Naja, einen gab es schon: Papas Chef. Dieser eitle Typ, der
+Jemanden anderen? Na ja, einen gab es schon: Papas Chef. Dieser eitle Typ, der
@@ -117,1 +117,1 @@
-Fluchvorbereitungen gewusst zu haben.
+Fluchtvorbereitungen gewusst zu haben.
@@ -122,3 +122,4 @@
-Wir planten zwei Anrufe. Der erste, wenn ich sicher die IMO verlassen habe – da
-sollten wir Radmer erzählen, dass er nominiert ist. Der zweite Anruf, wenn ich
-im Westen bin. Dann sollte er seinen Preis bekommen.
+Wir planten zwei Anrufe. Der erste sollte stattfinden, sobald ich die IMO sicher
+verlassen hatte – da sollten wir Radmer erzählen, dass er nominiert ist. Der
+zweite sollte folgen, wenn ich im Westen bin. Dann sollte er seinen Preis
+bekommen.
@@ -132,2 +133,2 @@
-französischen Nominierung prahlt, dann wissen meine Eltern, dass ich es raus
-geschafft habe. Und keiner kann's verhindern.“
+französischen Nominierung prahlt, dann wissen meine Eltern, dass ich es
+rausgeschafft habe. Und keiner kann's verhindern.“