--- old
+++ new
@@ -46,1 +46,1 @@
-Dresden -- der Typ konnte die Formeln aus dem Ärmel schütteln, wo die anderen
+Dresden – der Typ konnte die Formeln aus dem Ärmel schütteln, wo die anderen
@@ -52,1 +52,1 @@
-war nämlich so ein Parteihengst -- schon in der FDJ groß, dann früh in der SED.
+war nämlich so ein Parteihengst – schon in der FDJ groß, dann früh in der SED.
@@ -58,1 +58,1 @@
-Sicherheitsforschung' aufgemacht wurde -- rat mal, wer da Professor wurde? Klar,
+Sicherheitsforschung' aufgemacht wurde – rat mal, wer da Professor wurde? Klar,
@@ -61,2 +61,2 @@
-Nur: der Typ hatte null Plan. Völlig überfordert mit der ganzen Nummer. Also hat
-er meinen Vater als wissenschaftlichen Mitarbeiter geholt -- offiziell, um sich
+Nur: Der Typ hatte null Plan. Völlig überfordert mit der ganzen Nummer. Also hat
+er meinen Vater als wissenschaftlichen Mitarbeiter geholt – offiziell, um sich
@@ -69,1 +69,1 @@
-falsche.„
+falsche.“
@@ -76,1 +76,1 @@
-längst tot. Alle Welt lobt die ‚Arbeiten von Professor Radmer'. Aber eigentlich
+längst tot. Alle Welt lobt die ‚Arbeiten von Professor Radmer‘. Aber eigentlich
@@ -80,2 +80,2 @@
-die Maske.“\ Sie schaut mich schief an, als wollte sie prüfen, ob ich es auch so
-empfinde. „Maske sind leichter zu zerreißen als Herzen, weißt du?“
+die Maske.“ Sie schaut mich schief an, als wollte sie prüfen, ob ich es auch so
+empfinde. „Masken sind leichter zu zerreißen als Herzen, weißt du?“
@@ -89,1 +89,1 @@
-Ich kicher, obwohl meine Gedanken noch bei meinem Vater hängen.
+Ich kichere, obwohl meine Gedanken noch bei meinem Vater hängen.