--- old
+++ new
@@ -59,1 +59,1 @@
-„Ja, kapiert! Ich war echt stolz auf meine Pünktllichkeit!“ protestierte ich.
+„Ja, kapiert! Ich war echt stolz auf meine Pünktlichkeit!“, protestierte ich.
@@ -61,1 +61,1 @@
-„Und dann bedenke, Du wirst an dem Tag viel zu viel Gepäck haben. Verdächtig,
+„Und dann bedenke, du wirst an dem Tag viel zu viel Gepäck haben. Verdächtig,
@@ -79,1 +79,1 @@
-Verkleiden auf’m Klo, zack, fertig. Aber jetzt klang’s ernst.
+Verkleiden auf’m Klo, zack, fertig. Aber jetzt klang es ernst.
@@ -100,2 +100,2 @@
-achten nur auf Köpfe. Die suchen die Schleifen und Bänder, nicht ’nen schwarz
-gefärbten Jungenkopf.“
+achten nur auf Köpfe. Die suchen die Schleifen und Bänder, nicht ’nen
+schwarzgefärbten Jungenkopf.“
@@ -137,1 +137,1 @@
-immer mit riesiger Tasche. Politisch aber zuverlässig…. …und trage schwarze
+immer mit riesiger Tasche. Politisch aber zuverlässig… und ich trage schwarze
@@ -146,1 +146,1 @@
-lang du die tragen musst.“
+lange du die tragen musst.“
@@ -152,1 +152,1 @@
-was haben wir bei deiner Erziehung bloß falsch gemacht?“, lachte er.
+Was haben wir bei deiner Erziehung bloß falsch gemacht?“, lachte er.
@@ -164,1 +164,1 @@
-wird sie sterben – und ich werd sie keine Sekunde vermissen.
+wird sie sterben – und ich werde sie keine Sekunde vermissen.