--- old
+++ new
@@ -51,3 +51,3 @@
-Er mustert mich dauernd, als wäre ich irgendein Rätsel, das er unbedingt lösen
-will. Ich tue so, als würde ich’s nicht merken, auch wenn mir jedes Mal die
-Nackenhaare hochgehen.
+Er mustert mich aber immer noch so, als wäre ich irgendein Rätsel, das er
+unbedingt lösen will. Ich tue so, als würde ich’s nicht merken, auch wenn mir
+jedes Mal die Nackenhaare hochgehen.
@@ -133,1 +133,1 @@
-dass euer Kapitalismus zerbricht.“
+wie der Kapitalismus an sich selbst zerbricht und der Sozialismus triumphiert.“