--- old
+++ new
@@ -193,1 +193,1 @@
-ziehe mein Hemd aus, wringe, hänge es an einen rostigen Nagel – es tropft
+ziehe mein Hemd aus, wringe es aus, hänge es an einen rostigen Nagel – es tropft
@@ -201,3 +201,3 @@
-fühle mich wie ausgeliefert, kleiner als klein. Ich esse kalte Nudeln aus der
-Tüte, versuche nicht zu heulen, doch die Tränen laufen sowieso. In dem Moment,
-verfluche ich in den Regen, als wäre er ein lebendiger Feind.
+fühle mich ausgeliefert, kleiner als klein. Ich esse kalte Nudeln aus der Tüte,
+versuche nicht zu heulen, doch die Tränen laufen sowieso. In dem Moment
+verfluche ich den Regen, als wäre er ein lebendiger Feind.
@@ -214,1 +214,1 @@
-Die darauf folgende Hütte, Sompalaavu, liegt in einem alten Torfmoor. Gräben
+Die darauffolgende Hütte, Sompalaavu, liegt in einem alten Torfmoor. Gräben
@@ -217,5 +217,5 @@
-winddurchlässig. Ich versuche ein Feuer, es ertrinkt sofort. Kalte Nudeln,
-kalter Atem. Nachts die Schüttelfröste, mein ganzer Körper schlägt im Takt des
-Regens. Ich liege da und weiß: ich habe die Kontrolle verloren. Ich denke an den
-See am Anfang, an das Gefühl, unbesiegbar zu sein. Und könnte fast lachen, so
-weit weg ist das.
+winddurchlässig. Ich versuche, ein Feuer zu machen. Es ertrinkt sofort. Kalte
+Nudeln, kalter Atem. Nachts die Schüttelfröste, mein ganzer Körper schlägt im
+Takt des Regens. Ich liege da und weiß: Ich habe die Kontrolle verloren. Ich
+denke an den See am Anfang, an das Gefühl, unbesiegbar zu sein. Und könnte fast
+lachen, so weit weg ist das.
@@ -237,1 +237,1 @@
-dorthin ab. Ich bleibe stehen, kämpfe mit mir, übe im Kopf einen Satz, stark,
+dorthin ab. Ich bleibe stehen, kämpfe mit mir, übe im Kopf einen Satz: stark,
@@ -244,1 +244,1 @@
-Säure zerfressen. Ich schmiere sie trocken, es nützt nichts. Ich breite meine
+Säure zerfressen. Ich reibe sie trocken, es nützt nichts. Ich breite meine
@@ -246,1 +246,1 @@
-Laib Brot und ein ordentliches Stück Käse. Ich will „danke“ sagen, doch meine
+Laib Brot und ein ordentliches Stück Käse. Ich will „Danke“ sagen, doch meine