--- old
+++ new
@@ -47,2 +47,2 @@
-Ich bleibe noch ein Weilchen eingekuschelt im Schlafsack liegen und sehe Peter
-zu, wie er draußen Wasser geholt hat und jetzt mit seinem Kocher hantiert. Ganz
+Ich bleibe noch ein Weilchen eingekuschelt im Schlafsack liegen und sehe zu, wie
+Peter draußen Wasser geholt hat und jetzt mit seinem Kocher hantiert. Ganz
@@ -56,2 +56,2 @@
-Ich zucke zusammen, eröte. „Ich? Nein… also… ich finde nur… …also echt, diese
-unverschämt schönen Augen.“
+Ich zucke zusammen, erröte. „Ich? Nein ... also ... ich finde nur ... also echt,
+diese unverschämt schönen Augen.“
@@ -62,1 +62,1 @@
-Conny kichert. „Genau. Aber nur weil ich bei diesem Muskelberg nicht punkten
+Conny kichert. „Genau. Aber nur, weil ich bei diesem Muskelberg nicht punkten
@@ -76,2 +76,2 @@
-ich mich nicht wie eine Fremde. Ich spüre, mein Geheimnis ist bei ihnen sicher.
-Es ist, als würde von jedem von ihnen ein kleines Stück Wärme zu mir
+ich mich nicht wie eine Fremde. Ich spüre, dass mein Geheimnis bei ihnen sicher
+ist. Es ist, als würde von jedem von ihnen ein kleines Stück Wärme zu mir
@@ -81,2 +81,2 @@
-an -- keiner hat Lust, klatschnass zu paddeln oder durch den Wald zu stapfen.
-„Wir bleiben noch eine Nacht“, verkündet Conny. „Machen's uns gemütlich und
+an; keiner hat Lust, klatschnass zu paddeln oder durch den Wald zu stapfen. „Wir
+bleiben noch eine Nacht“, verkündet Conny. „Machen wir's uns gemütlich und
@@ -86,4 +86,3 @@
-Kocher fauchen, das Gelächter der anderen, und ich denke: *Wenn das schon
-Freiheit ist -- wie wird dann erst mein richtiges Leben aussehen, wenn ich es
-wirklich geschafft habe?*
-
+Kocher fauchen, das Gelächter der anderen und denke: *Wenn das schon Freiheit
+ist, wie wird dann erst mein richtiges Leben aussehen, wenn ich es wirklich
+geschafft habe?*