--- old
+++ new
@@ -9,1 +9,1 @@
-**Kurz:** Die Wahl fiel bewusst auf Mathematik: Sie hat autobiografische Wurzeln – der Autor selbst sah die Internationale Mathematik-Olympiade einst als möglichen Ausweg aus der DDR. Zugleich wird Mathe hier zum Symbol: eine universelle Sprache jenseits von Ideologien und ein Werkzeug, mit dem Eddie ihr Überleben kalkuliert, Chancen erkennt und Wege in die Freiheit findet.
+---
@@ -11,0 +11,2 @@
+**Kurz:**  Die Wahl fiel bewusst auf Mathematik: Sie hat autobiografische Wurzeln – der Autor selbst sah die Internationale Mathematik-Olympiade einst als möglichen Ausweg aus der DDR. Zugleich wird Mathe hier zum Symbol: eine universelle Sprache jenseits von Ideologien und ein Werkzeug, mit dem Eddie ihr Überleben kalkuliert, Chancen erkennt und Wege in die Freiheit findet.
+