--- old
+++ new
@@ -70,1 +70,1 @@
-„Pass auf dich auf, Eddie.“ Für Mielke war’s ein banaler Satz, wie „zieh dich
+„Pass auf dich auf, Eddie.“ Für Mielke war’s ein banaler Satz, wie „Zieh dich
@@ -89,3 +89,3 @@
-Im Gang zum Boarding alles wie in einem Traum. Leute mit Plastiktüten, manche
-quasselten, andere starrten leer vor sich hin. Ich spürte jeden Schritt. Und
-dann knallte es mir plötzlich rein: Ich würde sie nicht wiedersehen. Nicht
+Im Gang zum Boarding war alles wie in einem Traum. Leute mit Plastiktüten,
+manche quasselten, andere starrten leer vor sich hin. Ich spürte jeden Schritt.
+Und dann knallte es mir plötzlich rein: Ich würde sie nicht wiedersehen. Nicht
@@ -96,2 +96,2 @@
-Deckgeschichten. Aber nicht an das Loch, das sie in mir hinterlassen würden.
-Nicht an das Risiko, dem ich sie aussetze. Man wird sie verhören, ihnen was
+Deckgeschichten. Aber nicht an das Loch, das sie in mir hinterließen. Nicht an
+das Risiko, dem ich sie aussetzte. Man wird sie verhören, ihnen was
@@ -106,1 +106,1 @@
-die Hand, nur ganz kurz, fast unsichtbar. Stell mir vor, sie taten dasselbe.
+die Hand, nur ganz kurz, fast unsichtbar. Stellte mir vor, sie taten dasselbe.
@@ -117,1 +117,1 @@
-Raija sagt nichts, lange. Atmet tief. Ihre Augen glänzen. Dann rückt sie näher,
+Raija sagt lange nichts. Atmet tief. Ihre Augen glänzen. Dann rückt sie näher,