--- old
+++ new
@@ -51,1 +51,1 @@
-nachzumachen, und natürlich kam bei mir erstmal gar nichts. Wochenlang. Funken
+nachzumachen und natürlich kam bei mir erstmal gar nichts. Wochenlang. Funken
@@ -66,1 +66,1 @@
-man sie anlehnt, welche Seite zum Wind drehen, wie man den Boden mit Laub
+man sie anlehnt, welche Seite man zum Wind dreht, wie man den Boden mit Laub
@@ -81,1 +81,1 @@
-und rechneten aus, wo Osten ist. Er machte daraus Geometrieaufgaben, mit Winkeln
+und rechneten aus, wo Osten ist. Er machte daraus Geometrieaufgaben mit Winkeln
@@ -90,1 +90,1 @@
-dass ich mich so gut in unserem Garten verstecken konnte. Vater machte ernst
+dass ich mich so gut in unserem Garten verstecken konnte. Vater machte Ernst
@@ -105,1 +105,1 @@
-ich. Ich lag da, lauschte auf jedes Knacken, und war sicher, dass hinter jedem
+ich. Ich lag da, lauschte auf jedes Knacken und war sicher, dass hinter jedem
@@ -131,1 +131,1 @@
-Ich sammele Pilze, Heidelbeeren, Holz. Als das Feuer knistert, ist es wie damals
+Ich sammle Pilze, Heidelbeeren, Holz. Als das Feuer knistert, ist es wie damals