--- old
+++ new
@@ -17,15 +17,2 @@
-  "feedback": {
-    "anni": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "lucia": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "soek": {
-      "rating": ".....",
-      "comments": ""
-    }
-  },
-  "todo": "Die Szene ist zu früh eingeordnet, sie muss hinter das Kennenlernen, aber vor die Duschszene"
+  "todo": "Die Szene ist zu früh eingeordnet, sie muss hinter das Kennenlernen, aber vor die Duschszene",
+  "proofreading": {}