--- old
+++ new
@@ -79,1 +79,1 @@
-nicke, leise: „Danke. Wirklich … danke.“ Und für einen Moment träume ich mich
+nicke leise. „Danke. Wirklich … danke.“ Und für einen Moment träume ich mich
@@ -84,1 +84,1 @@
-Als es dann wirklich soweit ist, tuscheln die vier hinter meinem Rücken,
+Als es dann wirklich so weit ist, tuscheln die vier hinter meinem Rücken,
@@ -88,1 +88,1 @@
-Damenhygiene, Kosmetik und Schokolade. Westliche. Sachen, die ich noch nie
+Damenhygiene, Kosmetik und Schokolade. Westliches. Sachen, die ich noch nie