--- old
+++ new
@@ -43,0 +43,3 @@
+> **➡ Spieltheorie und Entscheidungsbäume...**\
+> (Das ging jetzt alles ein bischen zu schnell. Die genaue Erläuterung findest du im Kapitel 2 von „Eddie rechnet“.)
+
@@ -45,1 +48,1 @@
-Okay, inzwischen hab ich Kronen, das Währungsproblem wäre also vom Tisch. Aber ehrlich: Die Variante bleibt trotzdem ein Totalausfall.
+Okay, inzwischen hab ich Finnische Markka, das Währungsproblem wäre also vom Tisch. Aber ehrlich: Die Variante bleibt trotzdem ein Totalausfall.