--- old
+++ new
@@ -38,3 +38,3 @@
-betrunken, die Augen glasig, der Atem stinkt nach billigem Schnaps. „Na,
-Eddie..." lallt er und kommt auf mich zu, schwankend. Er grinst schief. „Du bist
-viel hübscher als Raija..." Dann beugt er sich vor, versucht, mich zu küssen.
+betrunken, die Augen glasig, der Atem stinkt nach billigem Schnaps. „Na, Eddie…"
+lallt er und kommt auf mich zu, schwankend. Er grinst schief. „Du bist viel
+hübscher als Raija…" Dann beugt er sich vor, versucht, mich zu küssen.