--- old
+++ new
@@ -1,1 +1,1 @@
-# Jorma vergisst sicht
+# Jorma vergisst sich
@@ -69,1 +69,1 @@
-Befreiungsschlag. Draußen atme ich tief ein -- kalte Luft, klare Luft. Ich
+Befreiungsschlag. Draußen atme ich tief ein – kalte Luft, klare Luft. Ich
@@ -86,1 +86,1 @@
-herauskommen. Ihre Gesichter sind finster. Tja Eddie ist schon fort! Das war
+herauskommen. Ihre Gesichter sind finster. Tja, Eddie ist schon fort! Das war
@@ -97,1 +97,1 @@
-gefallen in dieser mordernen Stadt. Vorne hängt eine große Uhr, die allen
+gefallen in dieser modernen Stadt. Vorne hängt eine große Uhr, die allen
@@ -105,1 +105,0 @@
-