--- old
+++ new
@@ -49,3 +49,3 @@
-vor der Fähre zu ergattern, liefen immer nach demselben Schema ab:\ Ich klopfte
-an Türen oder Fenster der Fahrer, stand da wie ein Häufchen Elend -- ja, ich
-weiß, genau das, was Sini immer zu verhindern versucht -- und stammelte auf
+vor der Fähre zu ergattern, liefen immer nach demselben Schema ab: Ich klopfte
+an Türen oder Fenster der Fahrer, stand da wie ein Häufchen Elend – ja, ich
+weiß, genau das, was Sini immer zu verhindern versucht – und stammelte auf
@@ -56,3 +56,2 @@
-sei das Flirten ein Handwerk, das man in drei Abenden erlernen könne -- und
-nicht eine Kunst, für die ich schlicht das falsche Betriebssystem installiert
-habe.
+sei das Flirten ein Handwerk, das man in drei Abenden erlernen könne – und nicht
+eine Kunst, für die ich schlicht das falsche Betriebssystem installiert habe.
@@ -62,5 +61,5 @@
-aus der Werbung für irgendwas, was es nur in Schweden gibt, und flirtete eine
-halbe Ewigkeit, bevor sie die entscheidende Frage stellte. In beiden Fällen:
-keine Karten übrig. Aber die Fahrer sahen aus, als hätten sie für Sini am
-liebsten einen anderen Fahrgast aus dem Bus geworfen.\ „Tja, diesmal hatten wir
-Pech“, sagte Sini -- und grinste, als sei das alles nur ein Aufwärmspiel.
+aus der Werbung für etwas, das es nur in Schweden gibt, und flirtete eine halbe
+Ewigkeit, bevor sie die entscheidende Frage stellte. In beiden Fällen: keine
+Karten übrig. Aber die Fahrer sahen aus, als hätten sie für Sini am liebsten
+einen anderen Fahrgast aus dem Bus geworfen. „Tja, diesmal hatten wir Pech“,
+sagte Sini – und grinste, als sei das alles nur ein Aufwärmspiel.
@@ -69,2 +68,2 @@
-Ende die große Zahl an Versuchen. Und ich hatte in den letzten Wochen -- ob
-bewusst oder aus Bequemlichkeit -- verdammt viele Versuche einfach verfallen
+Ende die große Zahl an Versuchen. Und ich hatte in den letzten Wochen – ob
+bewusst oder aus Bequemlichkeit – verdammt viele Versuche einfach verfallen
@@ -72,1 +71,1 @@
-ernst und mache ich es für Sini. Und wenn ich mich für Sini ins Zeug lege, dann
+ernst und mache es für Sini. Und wenn ich mich für Sini ins Zeug lege, dann
@@ -76,1 +75,1 @@
-angeboten, mir etwas „Heißes“ -- Originalzitat -- auszusuchen. Heute bin ich
+angeboten, mir etwas „Heißes“ – Originalzitat – auszusuchen. Heute bin ich
@@ -81,2 +80,2 @@
-ausgewählt.\ „Sini, wenn ich in diesen Bus einsteige, kann ich dir die Sachen
-nicht zurückgeben.“\ „Es wäre mir eine Freude, wenn du sie behältst! Aber jetzt
+ausgewählt. „Sini, wenn ich in diesen Bus einsteige, kann ich dir die Sachen
+nicht zurückgeben.“ „Es wäre mir eine Freude, wenn du sie behältst! Aber jetzt
@@ -85,1 +84,1 @@
-Sie drückt mir etwas in die Hand, das kaum größer ist als ein Bikinioberteil.\
+Sie drückt mir etwas in die Hand, das kaum größer ist als ein Bikinioberteil.
@@ -88,2 +87,2 @@
-als würde sie demonstrativ wegschauen.\ „Kannst gucken!\“, sage ich, halb
-trotzig, halb neugierig.
+als würde sie demonstrativ wegschauen. „Kannst gucken!“, sage ich, halb trotzig,
+halb neugierig.
@@ -91,2 +90,2 @@
-Und ja, das Outfit funktioniert -- leider nicht auf diese schwesterliche Art,
-wie ich's mir von Sini wünschen würde. Sie tanzt um mich herum wie ein Kind, das
+Und ja, das Outfit funktioniert – leider nicht auf diese schwesterliche Art, wie
+ich's mir von Sini wünschen würde. Sie tanzt um mich herum wie ein Kind, das
@@ -94,2 +93,2 @@
-gutes Stück größer als Du, findest Du nicht, alles ist etwas… …kurz?“\ „Absolut
-Eddie, absolut! Was soll ich sagen -- du bist heiß!“
+gutes Stück größer als du, findest du nicht, alles ist etwas… kurz?“ „Absolut,
+Eddie, absolut! Was soll ich sagen – du bist heiß!“
@@ -100,1 +99,1 @@
-für mich immer das höchste Maß an Schönheit gewesen. Und jetzt -- komisch --
+für mich immer das höchste Maß an Schönheit gewesen. Und jetzt – komisch –
@@ -104,1 +103,1 @@
-nicht wieder etwas schrecklich Dummes zu tun.“\ Ich ziehe die Augenbrauen hoch.
+nicht wieder etwas schrecklich Dummes zu tun.“ Ich ziehe die Augenbrauen hoch.
@@ -109,1 +108,1 @@
-keinen Plan B und die Zeit läuft uns davon*.\ „Deine Haare sehen allerdings
+keinen Plan B und die Zeit läuft uns davon.* „Deine Haare sehen allerdings
@@ -118,1 +117,1 @@
-Sini vermutlich umwerfend aussieht -- und an mir… na ja.
+Sini vermutlich umwerfend aussieht – und an mir… na ja.
@@ -125,4 +124,4 @@
-Aber dann… ich weiß nicht, ob's das Top ist, der Rock oder einfach die Tatsache,
-dass ich beschlossen habe, *jetzt* zu gewinnen -- werde ich mutiger. Fange an zu
-lachen, zu kontern, scherzend von Bus zu Bus zu tingeln. Bislang jedoch kein
-freies Ticket.
+Aber dann … ich weiß nicht, ob's das Top ist, der Rock oder einfach die
+Tatsache, dass ich beschlossen habe, *jetzt* zu gewinnen – werde ich mutiger.
+Fange an zu lachen, zu kontern, scherzend von Bus zu Bus zu tingeln. Bislang
+jedoch kein freies Ticket.
@@ -141,1 +140,1 @@
-„Was soll ich sagen, heiße kleine Eddie -- ich bin so froh, dass du noch einen
+„Was soll ich sagen, heiße kleine Eddie – ich bin so froh, dass du noch einen
@@ -150,1 +149,1 @@
-„Finger weg, die gehört mir!“ Ein Scherz -- sie weiß, dass ich nie jemandem
+„Finger weg, die gehört mir!“ Ein Scherz – sie weiß, dass ich nie jemandem
@@ -168,1 +167,1 @@
-„Das ist süß von dir“, lächelt sie. „Komm, wir gehen zur unserer alten Bank am
+„Das ist süß von dir“, lächelt sie. „Komm, wir gehen zu unserer alten Bank am