--- old
+++ new
@@ -50,1 +50,1 @@
-„Hast du Lust, ein bisschen rauszugehen?“, fragt sie nach dem Frühstück.\ Ich
+„Hast du Lust, ein bisschen rauszugehen?“, fragt sie nach dem Frühstück. Ich
@@ -56,1 +56,1 @@
-Schokolade in hundert Sorten. Und die Läden -- offen, obwohl Sonntag ist! Ich
+Schokolade in hundert Sorten. Und die Läden – offen, obwohl Sonntag ist! Ich
@@ -78,1 +78,1 @@
-Republikflucht-Vorbereitung. Die waren völlig normal, total nett. Und zack –
+Republikfluchtvorbereitung. Die waren völlig normal, total nett. Und zack –
@@ -98,2 +98,2 @@
-Landesolympiade. Mehrfach Bezirkssiegerin. Aber das reichte nicht um bei der IMO
-dabei zu sein. Sie wollten mehr. Also: Spezialschule Berlin, Lehrgänge,
+Landesolympiade. Mehrfach Bezirkssiegerin. Aber das reichte nicht, um bei der
+IMO dabei zu sein. Sie wollten mehr. Also: Spezialschule Berlin, Lehrgänge,
@@ -122,1 +122,1 @@
-jahrelang in Hörsälen sitzen. Und…“ Sie zögert kurz. „In meiner Heimatstadt
+jahrelang in Hörsälen sitzen. Und …“ Sie zögert kurz. „In meiner Heimatstadt
@@ -142,1 +142,1 @@
-ein Bäcker -- immer um die nächste Ecke.“
+ein Bäcker – immer um die nächste Ecke.“
@@ -149,1 +149,0 @@
-