--- old
+++ new
@@ -16,14 +16,1 @@
-  "feedback": {
-    "anni": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "lucia": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "soek": {
-      "rating": ".....",
-      "comments": ""
-    }
-  }
+  "proofreading": {}
@@ -57,1 +44,1 @@
-nervös, fast feierlich. „Von meinen Eltern," sagt er leise.
+nervös, fast feierlich. „Von meinen Eltern,“ sagt er leise.
@@ -73,1 +60,1 @@
-Weihnachtsfest… herzlich eingeladen." sogar auf Deutsch! Ich lese es zweimal,
+Weihnachtsfest… herzlich eingeladen.“ sogar auf Deutsch! Ich lese es zweimal,
@@ -81,1 +68,1 @@
-dich wirklich dabei haben wollen."
+dich wirklich dabei haben wollen.“
@@ -85,1 +72,1 @@
-und weißer Tischdecken… und ich hab nicht mal ein Geschenk!"
+und weißer Tischdecken… und ich hab nicht mal ein Geschenk!“
@@ -92,1 +79,1 @@
-„Ich muss etwas für dich haben," platzt es aus mir heraus, meine Stimme zittert.
+„Ich muss etwas für dich haben,“ platzt es aus mir heraus, meine Stimme zittert.
@@ -94,1 +81,1 @@
-Geschenk."
+Geschenk.“
@@ -97,1 +84,1 @@
-das Geschenk, Eddie."
+das Geschenk, Eddie.“
@@ -101,1 +88,1 @@
-sonst gehöre ich nie richtig dazu."
+sonst gehöre ich nie richtig dazu.“
@@ -105,1 +92,1 @@
-Irgendwie… ich weiß nicht wie, aber ich finde was."
+Irgendwie… ich weiß nicht wie, aber ich finde was.“
@@ -108,1 +95,1 @@
-musst nichts beweisen. Nicht mir, nicht meinen Eltern."
+musst nichts beweisen. Nicht mir, nicht meinen Eltern.“
@@ -110,1 +97,1 @@
-Ich beiße mir auf die Lippe, schüttle den Kopf. „Doch. Mir selbst."
+Ich beiße mir auf die Lippe, schüttle den Kopf. „Doch. Mir selbst.“