--- old
+++ new
@@ -73,1 +73,1 @@
-Weihnachtsfest... herzlich eingeladen." sogar auf Deutsch! Ich lese es zweimal,
+Weihnachtsfest… herzlich eingeladen." sogar auf Deutsch! Ich lese es zweimal,
@@ -80,1 +80,1 @@
-Vidar legt vorsichtig die Hand auf meinen Arm. „Eddie... es bedeutet, dass sie
+Vidar legt vorsichtig die Hand auf meinen Arm. „Eddie… es bedeutet, dass sie
@@ -83,3 +83,3 @@
-Ich schüttle den Kopf, presse die Einladung an meine Brust. „Aber... wie soll
-das gehen? Ich habe doch nichts! Ich sitze da, an einem Tisch voller
-Silberbesteck und weißer Tischdecken... und ich hab nicht mal ein Geschenk!"
+Ich schüttle den Kopf, presse die Einladung an meine Brust. „Aber… wie soll das
+gehen? Ich habe doch nichts! Ich sitze da, an einem Tisch voller Silberbesteck
+und weißer Tischdecken… und ich hab nicht mal ein Geschenk!"
@@ -100,1 +100,1 @@
-es nicht sein. Ich will dir etwas geben. Dir und... deinen Eltern auch. Sonst...
+es nicht sein. Ich will dir etwas geben. Dir und… deinen Eltern auch. Sonst…
@@ -105,1 +105,1 @@
-Irgendwie... ich weiß nicht wie, aber ich finde was."
+Irgendwie… ich weiß nicht wie, aber ich finde was."