--- old
+++ new
@@ -76,1 +76,1 @@
-„Du bist eine besonders, murmelt Sini...
+„Du bist etwas Besonders“, murmelt Sini...
@@ -80,1 +80,1 @@
-„In der DDR gab’s diese Mathe-Spezialschulen. Die nächste mit Internat war in
+„In der DDR gibt’s diese Mathe-Spezialschulen. Die nächste mit Internat war in
@@ -97,1 +97,1 @@
-mir zu lassen. Der Direktor dort war ein braver Parteisoldat in Menschengestalt.
+mir zu lassen. Der Direktor dort war eine Parteibestie in Menschengestalt.
@@ -132,1 +132,1 @@
-schicken sollte. Und sie war sich nicht ganz sicher, ob sie das richtig fände.
+schicken sollte. Und sie war sich sicher, dass das nicht richtig wäre.
@@ -164,1 +164,1 @@
-sie einen kleinen proletarisch-akademischen Gipfel organisiert: den Dekan, den
+sie einen kleine...“, ich forme mit den Finger Gänsefüsschen „...proletarisch-akademischen Gipfel organisiert: den Dekan, den
@@ -184,1 +184,1 @@
-Dekan hat Zahlen aufgezählt—meine Punkte, Bezirksplätze, Empfehlungsschreiben.
+Dekan hat Zahlen aufgezählt — meine Punkte, Bezirksplätze, Empfehlungsschreiben.
@@ -198,1 +198,1 @@
-gesagt: ‚Bei uns rechnet man besser, wenn man nicht redet.‘“
+gesagt: ‚Bei uns rechnet man besser, wenn man schweigt.‘“