--- old
+++ new
@@ -54,1 +54,1 @@
-Wollte er am Ende, dass ich die Wohnung verlasse? Hatte er vielleicht sogar
+Wollte er am Ende, dass ich die Wohnung verlasse? Hatte er vielleicht sogar ein
@@ -62,1 +62,1 @@
-Tampere ist ein Riese. Kaum stolper ich aus dem Backstein-Klotz von Bahnhof,
+Tampere ist ein Riese. Kaum stolpere ich aus dem Backsteinklotz vom Bahnhof,
@@ -79,2 +79,2 @@
-hängen. Da ist ein Bücherstand, riecht nach Vertrautem und nach Abenteuer
-zugleich.
+hängen. Da ist ein Bücherstand, der nach Vertrautem und nach Abenteuer zugleich
+riecht.
@@ -109,2 +109,2 @@
-Koordinaten. Ich teste sofort: suche eine Hütte bei Tampere, gleiche die
-Koordinaten mit einer Karte ab – und tatsächlich: ein kleiner See, mit Häuschen
+Koordinaten. Ich teste sofort und suche eine Hütte bei Tampere, gleiche die
+Koordinaten mit einer Karte ab – und tatsächlich: ein kleiner See mit Häuschen
@@ -126,1 +126,2 @@
-Am Ende habe ich 190 Markka ausgegeben. Und halte meine Beute wie Pokal.
+Am Ende habe ich 190 Markka ausgegeben. Und ich halte meine Beute wie einen
+Pokal.
@@ -136,1 +137,2 @@
-Reis und zwei Schweine namens Nudeln. Aber nicht sein ganzes Geld ausgegeben.
+Reis und zwei Schweine namens Nudeln. Aber Bauer Eddie hat nicht sein ganzes
+Geld ausgegeben.
@@ -139,1 +141,1 @@
-> (Hätte ich all mein Geld exakt für Reis und Nudeln ausgeben, sähe die Rechnung wie im Kapitel 4 von „Eddie rechnet“ aus.)
+> (Hätte ich all mein Geld exakt für Reis und Nudeln auszugeben, sähe die Rechnung wie im Kapitel 4 von „Eddie rechnet“ aus.)
@@ -141,7 +143,7 @@
-Mit meinen Beuteln – Rucksack konnte ich mir nicht leisten – setze ich mich in
-den Schatten und sortiere meinen Schatz: Schlafsack, verbeultes Blechgeschirr,
-eine Regenjacke, ein Schweizer Taschenmesser mit kleiner Säge (noch nie gesehen,
-mega praktisch), zwei kleine Feuerzeuge, Mückenspray (überlebensnotwendig). Dazu
-die zwei Bücher, fünfzehn (!!) Wanderkarten im Maßstab 1:25.000, und ein
-Geschenk vom Buchhändler: ein winziges Finnisch–Englisch-Wörterbuch. Nicht übel
-für 220 Markka.
+Mit meinen Beuteln – einen Rucksack konnte ich mir nicht leisten – setze ich
+mich in den Schatten und sortiere meinen Schatz: Schlafsack, verbeultes
+Blechgeschirr, eine Regenjacke, ein Schweizer Taschenmesser mit kleiner Säge
+(noch nie gesehen, mega praktisch), zwei kleine Feuerzeuge, Mückenspray
+(überlebensnotwendig). Dazu die zwei Bücher, fünfzehn (!!) Wanderkarten im
+Maßstab 1:25.000 und ein Geschenk vom Buchhändler: ein winziges
+Finnisch–Englisch-Wörterbuch. Nicht übel für 220 Markka.