--- old
+++ new
@@ -122,3 +122,3 @@
-mega praktisch), Mückenspray (überlebensnotwendig). Dazu die zwei Bücher, fünfzehn (!!) Wanderkarten im Maßstab
-1:25.000, und ein Geschenk vom Buchhändler: ein winziges
-Finnisch–Englisch-Wörterbuch.
+mega praktisch), Mückenspray (überlebensnotwendig). Dazu die zwei Bücher,
+fünfzehn (!!) Wanderkarten im Maßstab 1:25.000, und ein Geschenk vom
+Buchhändler: ein winziges Finnisch–Englisch-Wörterbuch.