--- old
+++ new
@@ -55,1 +55,1 @@
-auf die Tanzfläche. Ich könnte noch „später vielleicht“ sagen wie beim ersten
+auf die Tanzfläche. Ich könnte noch „später vielleicht“ sagen, wie beim ersten
@@ -93,1 +93,1 @@
-an sich..
+an sich.
@@ -120,1 +120,1 @@
-Sini schaut mich an, halb verwundert, halb ertappt. „Wollest du nicht…?“ fragt
+Sini schaut mich an, halb verwundert, halb ertappt. „Wolltest du nicht…?“ fragt