--- old
+++ new
@@ -18,14 +18,1 @@
-  "feedback": {
-    "anni": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "lucia": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "soek": {
-      "rating": ".....",
-      "comments": ""
-    }
-  }
+  "proofreading": {}
@@ -65,1 +52,1 @@
-„Entschuldigen Sie… ich kenne Sie doch."
+„Entschuldigen Sie… ich kenne Sie doch.“
@@ -72,1 +59,1 @@
-Cornelia."
+Cornelia.“
@@ -77,1 +64,1 @@
-„Sie sind ihr Vater?" frage ich ungläubig.
+„Sie sind ihr Vater?“ frage ich ungläubig.
@@ -81,1 +68,1 @@
-einem halben Jahr hoffe ich, dass ich Sie treffe. Heute, endlich."
+einem halben Jahr hoffe ich, dass ich Sie treffe. Heute, endlich.“
@@ -86,1 +73,1 @@
-wollte mich nicht in Ihre Verabschiedung drängen. Das war Ihr Moment."
+wollte mich nicht in Ihre Verabschiedung drängen. Das war Ihr Moment.“
@@ -91,1 +78,1 @@
-wie Sie. Mathegenies. Wir könnten Sie gut gebrauchen."
+wie Sie. Mathegenies. Wir könnten Sie gut gebrauchen.“
@@ -102,1 +89,1 @@
-komme sehr gern!"
+komme sehr gern!“
@@ -107,1 +94,1 @@
-Osterferien begonnen - 3 Wochen!"
+Osterferien begonnen - 3 Wochen!“
@@ -112,1 +99,1 @@
-Er richtet sich auf, wünscht mir noch einen guten Flug und "bis dann gleich",
+Er richtet sich auf, wünscht mir noch einen guten Flug und „bis dann gleich“,
@@ -115,1 +102,1 @@
-das?"
+das?“
@@ -117,1 +104,1 @@
-Er wirkt erstaunt. „Man braucht eine Kreditkarte."
+Er wirkt erstaunt. „Man braucht eine Kreditkarte.“
@@ -119,1 +106,1 @@
-„Oh."
+„Oh.“
@@ -121,1 +108,1 @@
-„Wollen Sie telefonieren? So dringend, dass es nicht bis Bonn warten kann?"
+„Wollen Sie telefonieren? So dringend, dass es nicht bis Bonn warten kann?“
@@ -123,2 +110,2 @@
-Ich schaue auf die Uhr. 17:25, das passt genau. „Ja," sage ich. „Furchtbar
-dringend."
+Ich schaue auf die Uhr. 17:25, das passt genau. „Ja,“ sage ich. „Furchtbar
+dringend.“