--- old
+++ new
@@ -23,1 +23,1 @@
-Wieder steht dieser schicke Volvo-Bus vor’m Tor von Joutsenlampi. Bis Joutsa sind’s nur fünfzehn Minuten. Und klar, wir – die ostdeutsche Vorzeigetruppe – sind natürlich wieder viel zu früh da.\
+Schon steht da dieser schicke Volvo-Bus vor’m Tor von Joutsenlampi. Bis Joutsa sind’s nur fünfzehn Minuten. Und klar, wir – die ostdeutsche Vorzeigetruppe – sind natürlich wieder viel zu früh da.\