--- old
+++ new
@@ -12,1 +12,1 @@
-    "Mattie",
+    "Matti",
@@ -15,1 +15,1 @@
-  "synopsis": "Eddie bekommt von Sabine Tampons angeboten, probiert sie zum ersten Mal aus und entdeckt am sonnigen Morgen nach Regentagen beim gemeinsamen Baden im See eine neue Freiheit. Die Jungs Mattie und Peter zeigen auffälliges Interesse, was Eddie verlegen macht, aber insgeheim freut sie sich.",
+  "synopsis": "Eddie bekommt von Sabine Tampons angeboten, probiert sie zum ersten Mal aus und entdeckt am sonnigen Morgen nach Regentagen beim gemeinsamen Baden im See eine neue Freiheit. Die Jungs Matti und Peter zeigen auffälliges Interesse, was Eddie verlegen macht, aber insgeheim freut sie sich.",
@@ -63,3 +63,3 @@
-Plötzlich halte ich ein Stück erwachsene Welt in der Hand. Mit der Packung fühle
-ich mich nicht mehr wie ein Mädchen – sondern wie jemand, der ernst genommen
-wird.
+Plötzlich halte ich ein Stück erwachsener Welt in der Hand. Mit der Packung
+fühle ich mich nicht mehr wie ein Mädchen – sondern wie jemand, der ernst
+genommen wird.
@@ -81,1 +81,1 @@
-ist klar: wir machen alle FKK.
+ist klar: Wir machen alle FKK.