--- old
+++ new
@@ -19,14 +19,1 @@
-  "feedback": {
-    "anni": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "lucia": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "soek": {
-      "rating": ".....",
-      "comments": ""
-    }
-  }
+  "proofreading": {}
@@ -65,1 +52,1 @@
-„Mensch Eddie, mit dem sitzt Du heute Abend im Flieger!", stuppst mich Vidar von
+„Mensch Eddie, mit dem sitzt Du heute Abend im Flieger!“, stuppst mich Vidar von
@@ -69,1 +56,1 @@
-Vidar meint: "Palme hatte wenig übrig für totalitäre Systeme."
+Vidar meint: „Palme hatte wenig übrig für totalitäre Systeme.“