--- old
+++ new
@@ -70,2 +70,2 @@
-im Drucker. Ein derber Fluch, Raijas Chef schlägt mit der Faust auf den Tisch
-neben dem Gerät.
+im Drucker. Ein derber Fluch entweicht ihm, und Raijas Chef schlägt mit der
+Faust auf den Tisch neben dem Gerät.
@@ -89,2 +89,2 @@
-„Sind sie schon da, um mich zu holen?“, frage ich und wieder steigt die Panik in
-mir hoch.
+„Sind sie schon da, um mich zu holen?“, frage ich, und wieder steigt die Panik
+in mir hoch.