--- old
+++ new
@@ -120,2 +120,2 @@
-Mit Ihrer Geburtsurkunde und den Passbildern können wir ihnen sofort einen
-Pass der BRD ausstellen, mit dem Sie dann auf die Fähre nach Deutschland kommen.
+Mit Ihrer Geburtsurkunde und den Passbildern können wir ihnen sofort einen Pass
+der BRD ausstellen, mit dem Sie dann auf die Fähre nach Deutschland kommen.
@@ -154,2 +154,2 @@
-ihnen liegen Dokumente. Mielke zeigt immer wieder auf eins dieser
-Dokumente. Der ältere Polizist verliert zunehmend die Nerven.
+ihnen liegen Dokumente. Mielke zeigt immer wieder auf eins dieser Dokumente. Der
+ältere Polizist verliert zunehmend die Nerven.
@@ -245,3 +245,3 @@
-was es bedeutet. Ich will es ablehnen, in ein paar Tagen bin ich doch schon bei der
-Botschaft, aber Maria besteht darauf. Ihre Augen funkeln entschlossen,
-als wollte sie sagen: *Du brauchst das mehr als ich.*
+was es bedeutet. Ich will es ablehnen, in ein paar Tagen bin ich doch schon bei
+der Botschaft, aber Maria besteht darauf. Ihre Augen funkeln entschlossen, als
+wollte sie sagen: *Du brauchst das mehr als ich.*