--- old
+++ new
@@ -136,4 +136,4 @@
-Männer, vermutlich dem Übersetzer, immer wieder heftig auf den Rücken
-schlägt. Er scheint mit seiner Übersetzung nicht zufrieden zu sein. Auf dem
-Tresen vor ihnen liegen Dokumente. Mielke zeigt auch immer wieder auf eins
-dieser Dokumente. Der ältere Polizist verliert zunehmend die Nerven.
+Männer, vermutlich dem Übersetzer, immer wieder heftig auf den Rücken schlägt.
+Er scheint mit seiner Übersetzung nicht zufrieden zu sein. Auf dem Tresen vor
+ihnen liegen Dokumente. Mielke zeigt auch immer wieder auf eins dieser
+Dokumente. Der ältere Polizist verliert zunehmend die Nerven.