--- old
+++ new
@@ -57,1 +57,1 @@
-„Jetzt erzähle mal in Ruhe“, sagt sie auf Englisch.
+„Jetzt erzähl mal in Ruhe“, sagt sie auf Englisch.
@@ -61,4 +61,4 @@
-weggelaufen ist. Alles, was ich vorbereitet habe, die ganzen Jahre Angst und
-Pläne – ich kriege das nicht raus. Es bleibt hängen. Ich stolpere über Worte,
-erzähle viel zu hastig und springe hin und her. Aber Raija hört zu, nickt, fragt
-nicht dazwischen.
+weggelaufen ist. Alles, was ich vorbereitet habe, die ganzen Jahre voller Angst
+und Pläne – ich kriege das nicht raus. Es bleibt hängen. Ich stolpere über
+Worte, erzähle viel zu hastig und springe hin und her. Aber Raija hört zu,
+nickt, fragt nicht dazwischen.
@@ -81,1 +81,1 @@
-wählt, dann lange spricht. Ich verstehe nur einzelne Worte: Reisepass…
+wählt, dann lange spricht. Ich verstehe nur einzelne Wörter: Reisepass…
@@ -87,2 +87,2 @@
-Blatt mit Fischers Handschrift. Ich falte die Rückseite nach außen, dass man das
-Stempelchen sehen kann. „Birth certificate“, flüstere ich und schiebe es ihr
+Blatt mit Fischers Handschrift. Ich falte die Rückseite nach außen, damit man
+das Stempelchen sehen kann. „Birth certificate“, flüstere ich und schiebe es ihr
@@ -144,4 +144,4 @@
-Draußen Mielke, sein graues Gesicht voller Wut, daneben der ältere Polizist, sie
-brüllen durcheinander. Ein Übersetzer kriegt Schläge auf den Rücken, Dokumente
-liegen auf dem Tresen. Ich drücke die Hände fest gegeneinander, meine Nägel
-graben sich in die Haut.
+Draußen steht Mielke, sein graues Gesicht voller Wut, daneben der ältere
+Polizist, sie brüllen durcheinander. Ein Übersetzer kriegt Schläge auf den
+Rücken, Dokumente liegen auf dem Tresen. Ich drücke die Hände fest
+gegeneinander, meine Nägel graben sich in die Haut.
@@ -150,3 +150,3 @@
-Ich reiße die Luft ein. Sie hält sie zwar nach unten, aber wie sie dasteht, mit
-beiden Händen, so fest, so entschlossen… ich verkrampfe mich, als würde gleich
-ein Schuss fallen.
+Ich ziehe scharf die Luft ein. Sie hält sie zwar nach unten, aber wie sie
+dasteht, mit beiden Händen, so fest, so entschlossen… ich verkrampfe mich, als
+würde gleich ein Schuss fallen.
@@ -176,1 +176,1 @@
-„Aber er ist nicht verwandt mit dir?“
+„Aber er ist nicht mit dir verwandt?“
@@ -193,2 +193,2 @@
-„Ja!“ Meine Stimme überschlägt sich. Ich versuche die Worte einzeln zu
-übersetzen: „Jugend… Jugend-Werk-Hof!“
+„Ja!“ Meine Stimme überschlägt sich. Ich versuche, die Worte einzeln zu
+übersetzen: „Jugend… Jugendwerkhof!“
@@ -203,2 +203,2 @@
-In ihrem Finnisch höre ich ihn das deutsche Wort „Jugendwerkhof!“ sagen und an
-mich: „In Thorgau, yes?“
+In ihrem Finnisch höre ich ihn das deutsche Wort „Jugendwerkhof!“ sagen und sich
+an mich wenden: „In Torgau, yes?“
@@ -206,1 +206,1 @@
-Ich bin total perplex. „Ja, Thorgau! Genau, woher…?“
+Ich bin total perplex. „Ja, Torgau! Genau, woher…?“
@@ -213,1 +213,1 @@
-Plotzlich sagt auf er auf holprigem Englisch: „He… he hits you?“
+Plötzlich sagt er auf holprigem Englisch: „He… he hits you?“
@@ -221,2 +221,2 @@
-Ich starre sie an. Alles rauscht. „Oh…“ Entkommt es mir, überrascht, wie einfach
-das klingen soll. Eine kleine Lüge – die mich retten könnte?
+Ich starre sie an. Alles rauscht. „Oh…“, entkommt es mir, überrascht, wie
+einfach das klingen soll. Eine kleine Lüge – die mich retten könnte?