--- old
+++ new
@@ -120,1 +120,1 @@
-Mit Ihrer Geburtsurkunde und den Passbildern können wir ihnen dann sofort einen
+Mit Ihrer Geburtsurkunde und den Passbildern können wir ihnen sofort einen
@@ -154,1 +154,1 @@
-ihnen liegen Dokumente. Mielke zeigt auch immer wieder auf eins dieser
+ihnen liegen Dokumente. Mielke zeigt immer wieder auf eins dieser
@@ -210,1 +210,1 @@
-antwortet er: „Er schlägt dich!“
+antwortet er: „He hits you!“
@@ -245,2 +245,2 @@
-was es bedeutet. Ich will es ablehnen – morgen soll doch schon der Mann von der
-Botschaft kommen –, aber Maria besteht darauf. Ihre Augen funkeln entschlossen,
+was es bedeutet. Ich will es ablehnen, in ein paar Tagen bin ich doch schon bei der
+Botschaft, aber Maria besteht darauf. Ihre Augen funkeln entschlossen,