--- old
+++ new
@@ -6,0 +6,1 @@
+  "day": "Friday",
@@ -53,3 +54,3 @@
-einer großen Tasse Kakao zurück. Meine Hände zittern
-so sehr, dass ich fast Angst habe, die Tasse fallen zu lassen. Ich klammere mich
-an den Becher, als wäre er ein Rettungsring.
+einer großen Tasse Kakao zurück. Meine Hände zittern so sehr, dass ich fast
+Angst habe, die Tasse fallen zu lassen. Ich klammere mich an den Becher, als
+wäre er ein Rettungsring.
@@ -60,5 +61,4 @@
-nach mir. Ich klinge wie so ein Kind, das einfach von zu Hause
-weggelaufen ist. Alles, was ich vorbereitet habe, die ganzen Jahre Angst und
-Pläne – das kriege ich nicht erklärt. Ich stolpere über Worte,
-erzähle viel zu hastig und springe hin und her. Aber Raija hört zu, nickt, unterbricht mich
-nicht.
+nach mir. Ich klinge wie so ein Kind, das einfach von zu Hause weggelaufen ist.
+Alles, was ich vorbereitet habe, die ganzen Jahre Angst und Pläne – das kriege
+ich nicht erklärt. Ich stolpere über Worte, erzähle viel zu hastig und springe
+hin und her. Aber Raija hört zu, nickt, unterbricht mich nicht.
@@ -89,3 +89,3 @@
-Ich finde das Blatt mit Herrn Fischers Handschrift. Ich drehe den Block um und
-falte die Rückseite nach außen. „Birth certificate“, sage ich zu Raija und
-schiebe ihr den Block über den Tisch.
+Ich finde das Blatt mit Herrn Fischers Handschrift, drehe den Block um und falte
+die Rückseite nach außen. „Birth certificate“, sage ich zu Raija und schiebe ihr
+den Block über den Tisch.
@@ -119,4 +119,6 @@
-„Passen Sie auf! Ich schicke einen Angestellten nach Joutsa. Mal sehen, wie
-schnell das geht. Ich hoffe, dass er morgen, spätestens übermorgen bei Ihnen
-ist. Bis dahin müssen Sie in der Polizeistation bleiben. Ich bespreche das
-gleich noch mal mit Frau Rehtaaler – geben Sie sie mir bitte zurück.“
+„Wir haben leider keine Konsulate in der Nähe. Sie müssen nach Helsinki kommen.
+Mit Ihrer Geburtsurkunde und den Passbildern können wir ihnen dann sofort einen
+Pass der BRD ausstellen, mit dem Sie dann auf die Fähre nach Deutschland kommen.
+Allerdings steht das Wochenende an. Wir öffnen erst wieder am Montag. Bis dahin
+müssen Sie irgendwo anders unterkommen. Ich bespreche das gleich noch mal mit
+Frau Rehtaaler – geben Sie sie mir bitte zurück.“
@@ -195,1 +197,1 @@
-„Ja!“ Ich versuche es zu übersetzen: „Youth-Work-Yard“
+„Ja!“ Ich versuche den Begriff „Jugendwerkhof“ zu übersetzen: „Youth-Work-Yard“
@@ -204,1 +206,1 @@
-„Wenn psychische Gewalt nicht reicht, dann auch mit physischer.“
+„Wenn seelische Gewalt nicht reicht, dann auch mit körperlicher.“
@@ -220,5 +222,6 @@
-Beide nicken. Dann fährt Raija fort: „Dann machen wir das so: Wir warten auf den
-Vertreter der Botschaft, der morgen oder übermorgen kommt. Er wird dir einen
-Pass ausstellen und dich mit nach Helsinki nehmen und dich in die Fähre setzen.
-Wir werden die Zentrale informieren, dass wir dich in Gewahrsam haben, um dich
-vor deinem gewalttätigen Vormund zu beschützen.“
+Beide nicken. Dann fährt Raija fort: „Dann machen wir das so: Wir suchen Dir
+eine sichere Unterkunft bis Montag, dann fährst Du mit dem Bus nach Helsinki.
+Wenn die Stasi Probleme bereitet, halte Dich an die finnische Polizei. Die
+Botschaft wird dir einen Pass ausstellen und dich in die Fähre setzen. Wir
+werden die Zentrale informieren, dass wir dich in Gewahrsam haben, um dich vor
+deinem gewalttätigen Vormund zu beschützen.“
@@ -229,2 +232,2 @@
-hat mein Vater zu mir gesagt. Und ich glaube, Raija ist ein guter Mensch. Bei
-dem Gedanken schießen mir schon wieder die Tränen.
+hat mein Vater zu mir gesagt. Und ich glaube, Raija und Maria sind gute
+Menschen. Bei dem Gedanken schießen mir schon wieder die Tränen.