--- old
+++ new
@@ -8,1 +8,4 @@
-  "persons": ["Eddie", "Vidar"],
+  "persons": [
+    "Eddie",
+    "Vidar"
+  ],
@@ -26,1 +29,7 @@
-  "themes": ["Selbstbestimmung", "Ungleichheit", "Freude am Geben", "Ausbeutung", "Freundschaft"],
+  "themes": [
+    "Selbstbestimmung",
+    "Ungleichheit",
+    "Freude am Geben",
+    "Ausbeutung",
+    "Freundschaft"
+  ],
@@ -34,1 +43,2 @@
-  }
+  },
+  "proofreading": {}