--- old
+++ new
@@ -48,3 +48,3 @@
-Schon steht da dieser schicke Volvo-Bus vor’m Tor von Joutsenlampi. Bis Joutsa
-sind’s nur fünfzehn Minuten. Und klar, wir – die ostdeutsche Vorzeigetruppe –
-sind natürlich wieder viel zu früh da.\ Der Fahrer klappt freundlich die Türen
+Schon steht da dieser schicke Volvo-Bus vorm Tor von Joutsenlampi. Bis Joutsa
+sind's nur fünfzehn Minuten. Und klar, wir – die ostdeutsche Vorzeigetruppe –
+sind natürlich wieder viel zu früh da. Der Fahrer klappt freundlich die Türen
@@ -53,1 +53,1 @@
-„Hinten, letzte Reihe!“, brüllt einer der Jungs. Typisch.\ Heute ist auch Herr
+„Hinten, letzte Reihe!“, brüllt einer der Jungs. Typisch. Heute ist auch Herr
@@ -59,3 +59,3 @@
-Natürlich darf er uns nicht verraten was drankommt. Aber einer aus unserer
-Gruppe tuschelt, er hätte gehört, heute gäb’s ne Aufgabe zu unterbestimmten
-Gleichungssystemen. Diese Bauern-mit-Kühen-und-Schweinen-Dinger. Also nochmal
+Natürlich darf er uns nicht verraten, was drankommt. Aber einer aus unserer
+Gruppe tuschelt: Er hätte gehört, heute gäb's 'ne Aufgabe zu unterbestimmten
+Gleichungssystemen. Diese Bauern-mit-Kühen-und-Schweinen-Dinger. Also noch mal
@@ -64,1 +64,1 @@
-kann, sodass sein Geld genau aufgeht. Typische Aufgabe in der Matheolly.
+kann, sodass sein Geld genau aufgeht. Typische Aufgabe in der Mathe-Oly.
@@ -66,1 +66,1 @@
-Ich rolle innerlich die Augen. Glaube ich nie. Viel zu easy. Das ist Stoff für
+Ich rolle innerlich die Augen. Glaub ich nie. Viel zu easy. Das ist Stoff für
@@ -74,1 +74,1 @@
-„Ist doch billig“, platzt es aus mir raus. „Diophantische Gleichung!“\ Uff. Kam
+„Ist doch billig“, platzt es aus mir raus. „Diophantische Gleichung!“ Uff. Kam
@@ -78,1 +78,1 @@
-„Richtig, Eddie. Aber wissen es auch die anderen?“ Ein paar stöhnen.\ Und dann –
+„Richtig, Eddie. Aber wissen es auch die anderen?“ Ein paar stöhnen. Und dann –
@@ -81,1 +81,1 @@
-Leben!\ Mir bleibt das Herz stehen.
+Leben! Mir bleibt das Herz stehen.
@@ -86,1 +86,1 @@
-> **➡ Die diophantische Gleichung #1…**\
+> **➡ Die diophantische Gleichung #1…**
@@ -89,1 +89,1 @@
-Der Bus rollt los. Die anderen diskutieren ein bisschen, bis sie’s endlich
+Der Bus rollt los. Die anderen diskutieren ein bisschen, bis sie's endlich
@@ -93,3 +93,3 @@
-Block hin und her.\ Oh nein. Er merkt was.\ Er fährt mit den Fingern am Rand
-entlang, knickt ’ne Ecke, lässt die Seiten schnipsen. Dieses Geräusch – ich
-hör’s wie einen Schuss. Manche Seiten sind dicker. Zu dick. Er schaut mich an.
+Block hin und her. Oh nein. Er merkt was. Er fährt mit den Fingern am Rand
+entlang, knickt 'ne Ecke, lässt die Seiten schnipsen. Dieses Geräusch – ich
+hör's wie einen Schuss. Manche Seiten sind dicker. Zu dick. Er schaut mich an.
@@ -103,4 +103,4 @@
-Dann, ganz ruhig, deckt er die Ecke ab, klappt den Block zu.\ Reicht ihn mir
-zurück, so beiläufig, als wär nix. Schaut demonstrativ zum Fenster raus.\ Und
-sagt nach einer Weile, immer noch nach draußen:\ „Die diophantische Gleichung,
-Eddie, ist oft viel schwerer umzusetzen, als man denkt. Überschätz dich nicht.“\
+Dann, ganz ruhig, deckt er die Ecke ab, klappt den Block zu. Reicht ihn mir
+zurück, so beiläufig, als wär nix. Schaut demonstrativ zum Fenster raus. Und
+sagt nach einer Weile, immer noch nach draußen: „Die diophantische Gleichung,
+Eddie, ist oft viel schwerer umzusetzen, als man denkt. Überschätz dich nicht.“
@@ -117,3 +117,3 @@
-an den Betreuern vorbei.\ Fischer meidet meinen Blick. Ganz bewusst.\ Und da
-steht er auch wieder – Mielke. Sein Blick klebt an uns. Ich muss normal wirken.
-Ganz Edmilla, die Spaßkanone.
+an den Betreuern vorbei. Fischer meidet meinen Blick. Ganz bewusst. Und da steht
+er auch wieder – Mielke. Sein Blick klebt an uns. Ich muss normal wirken. Ganz
+Edmilla, die Spaßkanone.
@@ -128,2 +128,2 @@
-die Gruppe.\ Wenn er mich bisher nicht verraten hat, tut er’s jetzt auch nicht
-mehr.\ Unfreiwillig hab ich jetzt einen Mitwisser. Einen Mitverschwörer.\ Das
+die Gruppe. Wenn er mich bisher nicht verraten hat, tut er's jetzt auch nicht
+mehr. Unfreiwillig hab ich jetzt einen Mitwisser. Einen Mitverschwörer. Das