--- old
+++ new
@@ -33,3 +33,3 @@
-
-Ich sitze auf dem kleinen Sofa, die Knie an die Brust gezogen, als plötzlich die
-Tür aufgerissen wird. Die Stimme, die den Raum füllt, ist schneidend wie kalter
+Wieder einer dieser gemütlichen Abende. Wie so oft habe ich Sini vom Terminal abgeholt, quatschend und lachend sind wir über die lange Brücke in die Innenstadt und zu Sinis Wohnung geschlendert. 
+Nun sitze ich auf dem kleinen Sofa, die Knie an die Brust gezogen, genieße die Ruhe und Vertrauheit. Plötzlich wird die
+Tür aufgerissen. Die Stimme, die den Raum füllt, ist schneidend wie kalter
@@ -37,1 +37,1 @@
-Mappe fest unter dem Arm. Keine Frage, hier kündigt sich kein freundlicher
+Mappe fest unter dem Arm. Keine Frage, so kündigt sich kein freundlicher
@@ -42,2 +42,1 @@
-die Spannung liegt so dicht in der Luft, dass ich sie körperlich spüren kann.
-Die Stirn der Mutter ist hart gefurcht, ihre Augen blitzen, während Sini kleiner
+die Spannung liegt so dicht in der Luft, dass ich sie körperlich spüren kann. Ist es Sinis Mutter? Im Gegensatz zu Sini hat sie kein Sami Blut in ihren Adern, aber ihrem Temperament scheint das nicht zu schaden. Ihre Stirn ist hart gefurcht, ihre Augen blitzen, während Sini kleiner
@@ -67,1 +66,1 @@
-„Eddie...", flüstert Sini und faltet flehend die Hände. „Äiti, lopeta!“ –
+„Eddie...", flüstert Sini zu mir und faltet flehend die Hände. „Äiti, lopeta!“ –
@@ -105,1 +104,1 @@
-den Finger wickeln, morgen werde ich alles anwenden, was du mir gelehrt hast.
+den Finger wickeln, morgen werde ich alles anwenden, was du mich gelehrt hast.