--- old
+++ new
@@ -39,1 +39,1 @@
-erst ganz leise, und plötzlich höre ich die Worte: *„Die Gedanken sind frei..."*
+erst ganz leise, und plötzlich höre ich die Worte: *„Die Gedanken sind frei…"*
@@ -70,1 +70,1 @@
-„*Imagine all the people...*" -- sie singen auf Englisch, etwas unsicher, ein
+„*Imagine all the people…*" -- sie singen auf Englisch, etwas unsicher, ein
@@ -125,1 +125,1 @@
-„Du willst mich küssen...", Conny singt den Einstieg vorsichtig, und wir anderen
+„Du willst mich küssen…", Conny singt den Einstieg vorsichtig, und wir anderen
@@ -132,1 +132,1 @@
-„\...You want to kiss me..." -- wir bringen den Text über die Lippen, trotz des
+„\…You want to kiss me…" -- wir bringen den Text über die Lippen, trotz des