--- old
+++ new
@@ -89,1 +89,1 @@
-wieder Tee. Wenn’s keine Hütte gibt, strecke ich manchmal den Schlafsack einfach
+wieder Tee. Wenn's keine Hütte gibt, strecke ich manchmal den Schlafsack einfach
@@ -109,2 +109,2 @@
-– drei ältere Männer mit wettergegerbten Gesichtern und Angelruten über der
-Schulter. Freundlich, neugierig, ein bisschen schelmisch. Sie mustern mich, und
+– drei ältere Männer mit wettergegerbten Gesichtern und Angelruten über den
+Schultern. Freundlich, neugierig, ein bisschen schelmisch. Sie mustern mich, und
@@ -123,1 +123,1 @@
-wissen selbst sie hier, sind bestens ausgerüstet, mit dicken Rucksäcken und
+wissen selbst sie hier, sind bestens ausgerüstet mit dicken Rucksäcken und
@@ -127,2 +127,2 @@
-Beim Abschied dann ein kurzes stummes Sechs-Augengespräch zwischen ihnen, ein
-Nicken, und einer drückt mir einen Rucksack in die Hand – angeblich leer. Erst
+Beim Abschied dann: ein kurzes stummes Sechs-Augengespräch zwischen ihnen, ein
+Nicken – und einer drückt mir einen Rucksack in die Hand – angeblich leer. Erst
@@ -131,3 +131,3 @@
-auf, genieße sie mit einer Gier, die fast weh tut. Schokolade! Der süße
-Geschmack macht mich benommen. Ich schaue an mir herunter. Nach fünf Tagen
-Wildnis ist da ein Loch, wo mal ein Bauch war.
+auf, genieße sie mit einer Gier, die fast wehtut. Schokolade! Der süße Geschmack
+macht mich benommen. Ich schaue an mir herunter. Nach fünf Tagen Wildnis ist da
+ein Loch, wo mal ein Bauch war.