--- old
+++ new
@@ -72,2 +72,2 @@
-Am meisten leiden die Leute, die ein mehr bisschen Herz und ein bisschen mehr
-Hirn haben, als der Rest. So welche wie ich. Kein Wunder, dass ich ihn sofort
+Am meisten leiden die Leute, die ein bisschen mehr Herz und ein bisschen mehr
+Hirn haben als der Rest. So welche wie ich. Kein Wunder, dass ich ihn sofort
@@ -75,1 +75,1 @@
-Schüler gehört, die flüstern den Namen auch schon. Ich hab den nie laut gesagt!
+Schüler gehört, die den Namen auch schon flüstern. Ich hab den nie laut gesagt!