--- old
+++ new
@@ -36,1 +36,2 @@
-Als ich am nächsten Tag wieder auf der Tour bin, kommt mir eine Geschichte meiner Kindheit in den Sinn.
+Als ich am nächsten Tag wieder auf der Tour bin, kommt mir eine Geschichte
+meiner Kindheit in den Sinn.
@@ -61,4 +62,4 @@
-erschöpft schnallte erdie Skier unter seine Füße und machte sich auf den Weg. Sein
-Ziel war die Grenze nach Norwegen -- dort hoffte er auf Schutz, weit weg von den
-dänischen Soldaten. Der Wind biss in sein Gesicht und die endlosen Wälder
-schienen ihn verschlingen zu wollen. Doch Gustav war schneller als die
+erschöpft schnallte erdie Skier unter seine Füße und machte sich auf den Weg.
+Sein Ziel war die Grenze nach Norwegen -- dort hoffte er auf Schutz, weit weg
+von den dänischen Soldaten. Der Wind biss in sein Gesicht und die endlosen
+Wälder schienen ihn verschlingen zu wollen. Doch Gustav war schneller als die
@@ -93,2 +94,3 @@
-auch wieder Unterdrücker geworden? Haben wir nicht in der Schule gelernt, dass uns
-der Kommunismus der Sowjetunion vom Faschismus befreit hat? Und jetzt ringen wir erneut um Freiheit, Freiheit von unseren Befreiern?
+auch wieder Unterdrücker geworden? Haben wir nicht in der Schule gelernt, dass
+uns der Kommunismus der Sowjetunion vom Faschismus befreit hat? Und jetzt ringen
+wir erneut um Freiheit, Freiheit von unseren Befreiern?