--- old
+++ new
@@ -36,0 +36,2 @@
+Als ich am nächsten Tag wieder auf der Tour bin, kommt mir eine Geschichte meiner Kindheit in den Sinn.
+
@@ -91,3 +93,2 @@
-auch wieder Unterdrücker geworden? Haben wir nicht in der Schule gelernt, dass
-der Kommunismus uns vom Faschismus befreit hat? Und jetzt ringen wir erneut um
-Freiheit, Freiheit von unseren Befreiern?
+auch wieder Unterdrücker geworden? Haben wir nicht in der Schule gelernt, dass uns
+der Kommunismus der Sovijetunion vom Faschismus befreit hat? Und jetzt ringen wir erneut um Freiheit, Freiheit von unseren Befreiern?