--- old
+++ new
@@ -78,1 +78,1 @@
-Ich sehe mich um -- und da ist sie. Keine zehn Meter entfernt, an der Ecke, wo
+Ich sehe mich um – und da ist sie. Keine zehn Meter entfernt, an der Ecke, wo
@@ -99,2 +99,2 @@
-warum -- aber wir wissen es alle. Zu auffällig, zu nah am Kirchturm, zu leicht
-zu finden.
+warum – aber wir wissen es alle. Zu auffällig, zu nah am Kirchturm, zu leicht zu
+finden.
@@ -108,2 +108,2 @@
-gestern. Es knistert -- nicht vor Hoffnungslosigkeit, sondern vor einer Idee,
-die langsam Form annimmt.
+gestern. Es knistert – nicht vor Hoffnungslosigkeit, sondern vor einer Idee, die
+langsam Form annimmt.
@@ -112,1 +112,1 @@
-nehmen… und andere, die einer hübschen jungen Frau schon mal einen Gefallen
+nehmen … und andere, die einer hübschen jungen Frau schon mal einen Gefallen
@@ -114,1 +114,1 @@
-junge Frau“ in diesem Plan wohl wäre. Ich starre zurück, versuche nicht rot zu
+junge Frau“ in diesem Plan wohl wäre. Ich starre zurück, versuche, nicht rot zu
@@ -117,1 +117,1 @@
-„Und dann gibt's noch die Neuen“, sagt sie. „Die kennen die Regeln nicht so gut.
+„Und dann gibt’s noch die Neuen“, sagt sie. „Die kennen die Regeln nicht so gut.
@@ -122,1 +122,1 @@
-Busfahrer-Plan war Mist. Wir machen das sauberer. Zwei Tickets -- eins hin und
+Busfahrer-Plan war Mist. Wir machen das sauberer. Zwei Tickets – eins hin und
@@ -124,1 +124,1 @@
-bei einer Gefangenenüberführung.“\ „Klingt… offiziell“, sage ich und versuche,
+bei einer Gefangenenüberführung.“\ „Klingt … offiziell“, sage ich und versuche,
@@ -130,1 +130,1 @@
-Da fällt es mir ein. „Was, wenn wir… eins dieser Fax schicken?“\ Die beiden
+Da fällt es mir ein. „Was, wenn wir … eins dieser Faxe schicken?“\ Die beiden
@@ -134,1 +134,1 @@
-„Na du weißt ja Sachen“, lacht Sini. „Wir schreiben einen Auslieferungsauftrag,
+„Na, du weißt ja Sachen“, lacht Sini. „Wir schreiben einen Auslieferungsauftrag,
@@ -136,1 +136,1 @@
-schwedisch gut genug, Sini?“ Sini nickt.
+Schwedisch gut genug, Sini?“ Sini nickt.
@@ -141,1 +141,1 @@
-offiziellen Briefkopf, eine gefälschte Unterschrift, und sogar ein erfundenes
+offiziellen Briefkopf, eine gefälschte Unterschrift und sogar ein erfundenes