--- old
+++ new
@@ -6,0 +6,1 @@
+  "day": "Friday",
@@ -119,1 +120,1 @@
-darauf fängt die kleine Kirchenglocke im Turm an zu schlagen.
+darauf fängt die kleine Kirchenglocke im Turm neben der Kirche an zu schlagen.
@@ -129,1 +130,5 @@
-Die Männer schreien sich jetzt an.
+Die Männer schreien sich jetzt an. Laufen kreuz und quer durch die Kirche,
+während die kleine Glocke unbeirrt Marias Hilferuf in die Stadt hinaus schreit.
+Aber wird es jemandem auffallen? Eine Glocke, die am Nachmittag schlägt?
+Immerhin lenkt das Gebimmel die Männder von mir ab. Ein paar minuten Zeit
+gewonnen. Wofür?
@@ -131,3 +136,4 @@
-„Da hinten in der Ecke, da sind viele Schalter!“ „Manfred, komm mal rüber, hier
-steht irgendwas auf Finnisch!“ „Warte, das heißt Beleuchtung Chorgestühl… Ach
-hier: das könnten Glocken sein!“
+Nach einer halben Ewigkeit ruft einer der Männer plötzlich: „Da hinten in der
+Ecke, da sind viele Schalter!“ „René, komm mal rüber, hier steht irgendwas auf
+Finnisch!“ „Warte, das heißt Beleuchtung Chorgestühl… Ach hier: das könnten
+Glocken sein!“
@@ -135,1 +141,1 @@
-Ein Klick – und die Glocke verstummt nach zwei, drei Schlägen.
+Ein Klick – und die Glocke verstummt nach zwei, drei weiteren Schlägen.
@@ -170,1 +176,1 @@
-Maria kommt angerannt, doch einer der anderen Männer hält sie zurück.
+Maria kommt angerannt, doch einer der anderen Männer hält sie grob zurück.
@@ -180,1 +186,1 @@
-wenigen Stunden gescheitert.
+wenigen Minuten gescheitert.