--- old
+++ new
@@ -52,2 +52,2 @@
-„Nein!“, lacht sie, „komme aus Rautu in Karelien. Da bin ich aufgewachsen, meine
-Mutter war Russin, mein Vater Finne.“
+„Nein!“, lacht sie, „ich komme aus Rautu in Karelien. Da bin ich aufgewachsen,
+meine Mutter war Russin, mein Vater Finne.“
@@ -65,1 +65,1 @@
-„Ich war ein junges Mädchen so wie du“, sagt Maria.
+„Ich war ein junges Mädchen, so wie du“, sagt Maria.
@@ -96,1 +96,1 @@
-Wie viele sind es, frage ich mich.
+Wie viele sind es?, frage ich mich.
@@ -105,1 +105,1 @@
-„Ich wusste, dass sie mich verstehen! Wo ist sie?“
+„Ich wusste, dass Sie mich verstehen! Wo ist sie?“
@@ -139,1 +139,1 @@
-„Na Alte, das hat ja wohl nicht geklappt! Durchsucht die Kirche, sie wird uns
+„Na, Alte, das hat ja wohl nicht geklappt! Durchsucht die Kirche, sie wird uns
@@ -167,1 +167,1 @@
-..ein schneller Griff unter das Tischtuch an meinen Knöchel. Ein Mann zieht mich
+…ein schneller Griff unter das Tischtuch an meinen Knöchel. Ein Mann zieht mich
@@ -179,1 +179,1 @@
-Angst, das Sich-Verstellen, die blöden Kleidchen tragen – alles schon nach
+Angst, das Sichverstellen, die blöden Kleidchen tragen – alles schon nach
@@ -191,1 +191,1 @@
-Kirche, Poliisi steht auf ihren Unformen. Die Szene, die sie sehen, scheint
+Kirche, Poliisi steht auf ihren Uniformen. Die Szene, die sie sehen, scheint