--- old
+++ new
@@ -126,4 +126,4 @@
-Drinnen lehne ich am Schalter, sehe die Fähre entladen. Menschen, Taschen,
-Stimmen in allen Sprachen. Dann taucht sie auf: Sini, Jacke offen, die Haare
-zerzaust vom Wind. Sie entdeckt mich, lächelt und ruft: „Bin gleich fertig!“
-Dann kommt sie zu mir, die Schritte schnell, als wolle sie mich gleich
+Drinnen lehne ich am Schalter, sehe die Fähre entladen werden. Menschen,
+Taschen, Stimmen in allen Sprachen. Dann taucht sie auf: Sini, Jacke offen, die
+Haare zerzaust vom Wind. Sie entdeckt mich, lächelt und ruft: „Bin gleich
+fertig!“ Dann kommt sie zu mir, die Schritte schnell, als wolle sie mich gleich
@@ -144,1 +144,1 @@
-Knopf an der Bluse, der zufällig offensteht, ein Blick, der ein bisschen länger
+Knopf an der Bluse, der zufällig offen steht, ein Blick, der ein bisschen länger
@@ -152,1 +152,1 @@
-dir ein Ticket, mir ein freier Abend oder der Welt ein Stück mehr Gleichheit –
+dir ein Ticket, mir einen freien Abend oder der Welt ein Stück mehr Gleichheit –
@@ -158,1 +158,1 @@
-„Ach, Eddie.“ Sie stößt mich sanft an. „Das System ist schummeln. Wir gleichen
+„Ach, Eddie.“ Sie stößt mich sanft an. „Das System ist Schummeln. Wir gleichen
@@ -169,1 +169,2 @@
-Ich ziehe die Schultern hoch, weil ich ahne, was sie meint, aber mich sträubt.
+Ich ziehe die Schultern hoch, weil ich ahne, was sie meint, aber ich sträube
+mich.
@@ -177,1 +178,1 @@
-Ich rolle mit den Augen. „Einmal hab’ ich’s gemacht. Kam aber keine Junggeselle
+Ich rolle mit den Augen. „Einmal hab’ ich’s gemacht. Kam aber kein Junggeselle