--- old
+++ new
@@ -149,2 +149,2 @@
-Kurz darauf kam er mit einem Bündel Scheine zurück. „Im Kästchen waren 131
-Mark. Der Kurs lag zuletzt bei eins zu zwei-komma-zwei. Das macht rund freundlich aufgerundet 300
+Kurz darauf kam er mit einem Bündel Scheine zurück. „Im Kästchen waren 133
+Mark. Der Kurs lag zuletzt bei eins zu zwei-komma-zwei. Das macht freundlich aufgerundet 300
@@ -171,1 +171,1 @@
-„Zweihundert Kronen sind viel Geld, wenn man sie mit nach Hause nimmt. In der DDR könnte man davon vermutliche einige Monate leben. Aber hier in Finnland reicht es kaum für eine Zugfahrkarte und ein paar warme Mahlzeiten.“
+„Dreihundert Markka sind viel Geld, wenn man sie mit nach Hause nimmt. In der DDR könnte man davon vermutliche einige Monate leben. Aber hier in Finnland reicht es kaum für eine Zugfahrkarte und ein paar warme Mahlzeiten.“