--- old
+++ new
@@ -32,1 +32,1 @@
-Gelegenheit, meine paar D-Mark die ich hatte, in finnische Kronen zu tauschen. In meinem
+Gelegenheit, meine paar D-Mark die ich hatte, in finnische Markka zu tauschen. In meinem
@@ -100,0 +100,2 @@
+*Hier fehlt etwas*!!!
+
@@ -139,1 +141,1 @@
-sag mal, weißt du, wie ich die D-Mark in finnische Kronen tauschen kann?“
+sag mal, weißt du, wie ich die D-Mark in finnische Markka tauschen kann?“
@@ -148,1 +150,1 @@
-Mark. Der Kurs lag zuletzt bei eins zu eins-kommafünf. Das macht rund 200
+Mark. Der Kurs lag zuletzt bei eins zu zwei-komma-zwei. Das macht rund 200