--- old
+++ new
@@ -63,1 +63,1 @@
-Hinter mir ein Geräusch. Tropfen? Schritt?, die Hände werden feucht. Ich muss
+Hinter mir ein Geräusch. Tropfen? Schritte? Die Hände werden feucht. Ich muss
@@ -68,1 +68,1 @@
-trage -- hinten ist eine Lücke. „Verdamt! Das kostet mich wieder Zeit!“, fluche
+trage – hinten ist eine Lücke. „Verdammt! Das kostet mich wieder Zeit!“, fluche
@@ -74,1 +74,1 @@
-Hinterkopfpartie. Fast geschafft.. Diese bescheuerte sowjetische Vorzeige-Puppe
+Hinterkopfpartie. Fast geschafft... Diese bescheuerte sowjetische Vorzeige-Puppe
@@ -77,3 +77,3 @@
-Ich atme tief durch. Draußen stürmt ne Gruppe Jungs rein, palavern laut über die
-Aufgaben. Ich bin also noch nicht zu spät dran. Ich wisch mir die Stirn, klapp
-die Dose zu, steh auf.
+Ich atme tief durch. Draußen stürmt 'ne Gruppe Jungs rein, und sie palavern laut
+über die Aufgaben. Ich bin also noch nicht zu spät dran. Ich wisch mir die
+Stirn, klapp die Dose zu, steh auf.