--- old
+++ new
@@ -15,14 +15,1 @@
-  "feedback": {
-    "anni": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "lucia": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "soek": {
-      "rating": ".....",
-      "comments": ""
-    }
-  }
+  "proofreading": {}
@@ -33,1 +20,1 @@
-schon. „Statsminister Olof Palme sköts igår kväll…" -- erschossen. Mitten in der
+schon. „Statsminister Olof Palme sköts igår kväll…“ -- erschossen. Mitten in der
@@ -42,1 +29,1 @@
-Greta sagt immer wieder: „So etwas passiert nicht hier. Nicht in Schweden." Aber
+Greta sagt immer wieder: „So etwas passiert nicht hier. Nicht in Schweden.“ Aber
@@ -57,1 +44,1 @@
-„Du bist ganz bleich," sagt Vidar leise.
+„Du bist ganz bleich,“ sagt Vidar leise.
@@ -59,1 +46,1 @@
-„Vielleicht kenne ich ihn," flüstere ich. Meine Stimme klingt fremd. Und dann
+„Vielleicht kenne ich ihn,“ flüstere ich. Meine Stimme klingt fremd. Und dann
@@ -92,1 +79,1 @@
-Bundesrepublik Deutschland. Kein „Übermorgen", kein „bald", sondern ein klares
+Bundesrepublik Deutschland. Kein „Übermorgen“, kein „bald“, sondern ein klares