--- old
+++ new
@@ -44,1 +44,1 @@
-Und dann ballert immer wieder das Pokerspiel in meinen Kopf. Erst das Kribbeln vom Gewinn, dann dieser Schock auf dem Flur. Er. Mielke. Mein Herz stolperte, mir wurde heiß und kalt in einem. Allein von seinem Blick zieht sich jetzt noch mein Magen zusammen.
+Und dann ballert immer wieder das Pokerspiel in meinen Kopf. Erst das Kribbeln vom Gewinn, dann dieser Schock draußen im Halbdunkel zwischen den Hütten. Er. Mielke. Mein Herz stolperte, mir wurde heiß und kalt in einem. Allein von seinem Blick zieht sich jetzt noch mein Magen zusammen.
@@ -60,1 +60,1 @@
-Er schnaubte. „Nach dem Wettkampf? Einkaufszentrum?“
+Er schnaubte. „Nach dem Wettkampf? Ins Einkaufszentrum?“
@@ -64,1 +64,1 @@
-„Hm.“ Sein Gesicht blieb Beton. „Das überlege ich mir noch – versprochen ist erstmal nichts. Mir wäre lieber, du hättest die Mathematik im Kopf, um uns morgen einen ersten Platz zu erkämpfen. Statt Markka und Kleidchen.“
+„Hm.“ Sein Gesicht blieb Beton. „Das überlege ich mir noch – versprochen ist erstmal nichts. Mir wäre lieber, du hättest die Mathematik im Kopf, um uns morgen einen ersten Platz zu erkämpfen. Statt Kerle und Kleidchen.“