--- old
+++ new
@@ -96,3 +96,3 @@
-„Ein Kleidchen kaufen. So'n finnisches, wie die Mädchen hier tragen. Und
-vielleicht Eis oder Cola für die anderen.“ Ich hör' mich fast flehen: „Kann ich
-morgen in die Stadt? Bitte!“
+„Ein Kleid kaufen. So'n finnisches, wie die Mädchen hier tragen. Und vielleicht
+Eis oder Cola für die anderen.“ Ich hör' mich fast flehen: „Kann ich morgen in
+die Stadt? Bitte!“
@@ -104,1 +104,1 @@
-„Hm.“ Sein Gesicht blieb Beton. „Das überlege ich mir noch – versprochen ist
+„Hm.“ Sein Gesicht blieb wie Beton. „Das überlege ich mir noch – versprochen ist