--- old
+++ new
@@ -47,1 +47,1 @@
-Morgen, nach dem Wettkampf. Punkt. Kein Drumrum. Ich hab’s so geplant. Entweder
+Morgen, nach dem Wettkampf. Punkt. Kein Drumrum. Ich hab's so geplant. Entweder
@@ -49,1 +49,1 @@
-sie endlich abstreifen wie ’ne alte Haut. Und das ist meine einzige Gewissheit,
+sie endlich abstreifen wie 'ne alte Haut. Und das ist meine einzige Gewissheit,
@@ -54,1 +54,1 @@
-loszulassen. Ich lieg auf’m Bett, die Decke längst weggestrampelt, T-Shirt
+loszulassen. Ich liege auf'm Bett, die Decke längst weggestrampelt, T-Shirt
@@ -56,1 +56,1 @@
-Rascheln da. Als wär Schlafen das Einfachste der Welt.
+Rascheln da. Als wäre Schlafen das Einfachste der Welt.
@@ -58,5 +58,5 @@
-Bei mir? Fehlanzeige. Ich dreh mich von links nach rechts, wieder zurück. Kissen
-zu heiß, Laken nervt, Kopf noch schlimmer. Immer wieder Augen auf, ins
-Halbdunkel starren, das kein richtiges Dunkel ist. Durch die ollen Gardinen
-sickert dieses irre Licht der Mittsommernacht. Keine Nacht, kein Tag – nur so’n
-fieses Zwielicht, das mich auslacht: Haha, Eddie, nix Schlaf für dich! Dabei
+Bei mir? Fehlanzeige. Ich dreh' mich von links nach rechts, wieder zurück.
+Kissen zu heiß, Laken nervt, Kopf noch schlimmer. Immer wieder die Augen auf,
+ins Halbdunkel starren, das kein richtiges Dunkel ist. Durch die ollen Gardinen
+sickert dieses irre Licht der Mittsommernacht. Keine Nacht, kein Tag – nur so'n
+fieses Zwielicht, das mich auslacht: Haha, Eddie, kein Schlaf für dich! Dabei
@@ -70,2 +70,2 @@
-Ich starre an die Decke, Arme hinter’m Kopf verschränkt. Draußen ruft ein Vogel
-– mitten in der Nacht! Weiß wohl selber nicht, ob’s schon morgen ist. Genau mein
+Ich starre an die Decke, Arme hinterm Kopf verschränkt. Draußen ruft ein Vogel –
+mitten in der Nacht! Weiß wohl selber nicht, ob's schon morgen ist. Genau mein
@@ -82,1 +82,1 @@
-Sein Grinsen war schief. „Na Edda, hängst du jeden Abend mit dem Klassenfeind
+Sein Grinsen war schief. „Na, Edda, hängst du jeden Abend mit dem Klassenfeind
@@ -90,1 +90,1 @@
-„Dreißig Markka.“ Ich tat so, als wär’s nix. Er konnte unmöglich den genauen
+„Dreißig Markka.“ Ich tat so, als wär's nix. Er konnte unmöglich den genauen
@@ -95,2 +95,2 @@
-„Ein Kleidchen kaufen. So’n finnisches, wie die Mädchen hier tragen. Und
-vielleicht Eis oder Cola für die anderen.“ Ich hör mich fast flehen: „Kann ich
+„Ein Kleidchen kaufen. So'n finnisches, wie die Mädchen hier tragen. Und
+vielleicht Eis oder Cola für die anderen.“ Ich hör' mich fast flehen: „Kann ich
@@ -104,2 +104,2 @@
-erstmal nichts. Mir wäre lieber, du hättest die Mathematik im Kopf, um uns
-morgen einen ersten Platz zu erkämpfen. Statt Kerle und Kleidchen.“
+erst mal nichts. Mir wäre lieber, du hättest die Mathematik im Kopf, um uns
+morgen einen ersten Platz zu erkämpfen. Statt Kerlen und Kleidchen.“
@@ -107,2 +107,1 @@
-Dann drehte er sich weg und scheuchte mich mit ’ner Handbewegung ins Bett.
-
+Dann drehte er sich weg und scheuchte mich mit 'ner Handbewegung ins Bett.