--- old
+++ new
@@ -64,1 +64,1 @@
-nirgendwo sicher – nur hier bei Raija. Sie sie lässt mich nicht im Stich.
+nirgendwo sicher – nur hier bei Raija. Sie lässt mich nicht im Stich.
@@ -104,1 +104,1 @@
-Ich fühle mich ertappt und rechtfertige mich. „Nein, so ist es nicht ganz: viele
+Ich fühle mich ertappt und rechtfertige mich. „Nein, so ist es nicht ganz: Viele
@@ -125,2 +125,2 @@
-anders. Die Fassaden sind frisch gestrichen, die Gehwege sauber, sogar
-Blumenrabatten vor den Eingängen. Nichts wirkt halbfertig, es liegt kein
+anders. Die Fassaden sind frisch gestrichen, die Gehwege sauber, und vor den
+Eingängen liegen sogar Blumenrabatten. Nichts wirkt halbfertig, es liegt kein
@@ -140,2 +140,2 @@
-„Das ist Eddie“, sagt Raija knapp, so als wolle sie gleich alle Fragen abräumen.
-„Sie bleibt hier erstmal.“
+„Das ist Eddie“, sagt Raija knapp, so, als wolle sie gleich alle Fragen
+abräumen. „Sie bleibt hier erstmal.“
@@ -144,1 +144,1 @@
-einschätzen, ob er mich nett findet oder nicht, aber eins ist klar: mir ist er
+einschätzen, ob er mich nett findet oder nicht, aber eins ist klar: Mir ist er
@@ -160,1 +160,0 @@
-