--- old
+++ new
@@ -55,1 +55,1 @@
-Datum, den Wagen, die Nummer. Mein Herz klopft wie wild. „Danke, Raija... für
+Datum, den Wagen, die Nummer. Mein Herz klopft wie wild. „Danke, Raija… für
@@ -68,2 +68,2 @@
-Ich kann die Tränen kaum zurückhalten. „Ich... danke dir... für alles."\ „Du
-hast nichts zu danken", sagt sie und lächelt dabei so, dass die Wärme durch den
+Ich kann die Tränen kaum zurückhalten. „Ich… danke dir… für alles."\ „Du hast
+nichts zu danken", sagt sie und lächelt dabei so, dass die Wärme durch den