--- old
+++ new
@@ -119,1 +119,1 @@
-für Sini den breiten sächsischen Dialekt irgendwie in Englische über übertragen.
+für Sini den breiten sächsischen Dialekt irgendwie ins Englische zu übertragen.
@@ -136,1 +136,1 @@
-jede Silbe ,"eine Göttin der Hydrodynamik, noch Lyud-mila Kel-dysh,
+jede Silbe ,"eine russische Göttin der Hydrodynamik, noch Lyud-mila Kel-dysh,
@@ -143,1 +143,1 @@
-vor Lachen beinahe von der Bank. Es ist so schön mit Sina Zeit zu verlieren. Der
+vor Lachen beinahe von der Bank. Es ist so schön mit Sini Zeit zu verlieren. Der
@@ -145,1 +145,1 @@
-den Stoff meines verwaschenen roten Trainingsanzugs – ein Stück Dresden, das
+den Stoff meiner verwaschenen roten Trainingshose – ein Stück Dresden, das