--- old
+++ new
@@ -48,1 +48,1 @@
-Rauch -- irgendwo brennt wohl ein Lagerfeuer -- und nach den süßlich-harzigen
+Rauch – irgendwo brennt wohl ein Lagerfeuer – und nach den süßlich-harzigen
@@ -55,1 +55,1 @@
-Sache wäre. „Ach nein. Das heißt 'kleines blaues Mädchen'. Ich mag es nicht und
+Sache wäre. „Ach nein. Das heißt ‚kleines blaues Mädchen‘. Ich mag es nicht und
@@ -118,1 +118,1 @@
-„Eigentlich wollte mein Vater mich Ada nennen, nach Ada Lovelace“, erzähle ich.\
+„Eigentlich wollte mein Vater mich Ada nennen, nach Ada Lovelace“, erzähle ich.
@@ -125,4 +125,4 @@
-gab. Er wollte, dass ich eine Art Vorreiterin werde, so wie sie.“, sage ich.
-„Nur leider kam er nie zum Einsatz. Denn als er im Standesamt den Namen 'Ada' zu
-Protokoll gab, fragte die Standesbeamtin:\ 'Ada? Das machen wir nicht, den Namen
-haben wir hier noch nie vergeben. Wie kommen Sie denn auf sowas?'“ Ich versuchte
+gab. Er wollte, dass ich eine Art Vorreiterin werde, so wie sie“, sage ich. „Nur
+leider kam er nie zum Einsatz. Denn als er im Standesamt den Namen ‚Ada‘ zu
+Protokoll gab, fragte die Standesbeamtin: ‚Ada? Das machen wir nicht, den Namen
+haben wir hier noch nie vergeben. Wie kommen Sie denn auf so was?‘“ Ich versuche
@@ -130,7 +130,7 @@
-„'Ist eine britische Mathematikerin', soll mein Vater erwiderthaben.\ 'Ah, nee,
-die kenn ich nicht. Wollen Sie nicht 'ne sowjetische Mathematikerin nehmen,
-wenn's unbedingt eine sein soll?'\ 'Kennen Sie eine?'\ 'Nee, ehrlich gesagt
-nicht', gestand die Beamtin.\ 'Außerdem spricht man den Namen nicht 'Ada',
-sondern 'Eda' aus', wandte mein Vater ein.\ 'So wie Edda Göring?'\ 'Ja, naja,
-eigentlich eher lang.'\ 'Dann schreiben Sie doch gleich Edda, wenn Sie Edda
-meinen. Ada machen wir nicht.' Das war ihr letztes Wort.
+„‚Ist eine britische Mathematikerin‘, soll mein Vater erwidert haben. ‚Ah, nee,
+die kenn ich nicht. Wollen Sie nicht ’ne sowjetische Mathematikerin nehmen,
+wenn’s unbedingt eine sein soll?‘ ‚Kennen Sie eine?‘ ‚Nee, ehrlich gesagt
+nicht‘, gestand die Beamtin. ‚Außerdem spricht man den Namen nicht „Ada“,
+sondern „Eda“ aus‘, wandte mein Vater ein. ‚So wie Edda Göring?‘ ‚Ja, na ja,
+eigentlich eher lang.‘ ‚Dann schreiben Sie doch gleich Edda, wenn Sie Edda
+meinen. Ada machen wir nicht.‘ Das war ihr letztes Wort.
@@ -139,1 +139,1 @@
-gab es in der DDR auch eine Schauspielerin, die Edda hieß. Edda Dentges -- auf
+gab es in der DDR auch eine Schauspielerin, die Edda hieß. Edda Dentges – auf
@@ -143,14 +143,15 @@
-irgendwie schräg.“\ „Ja, oder? Aber hätte auch schlimmer kommen können weißt Du?
-Von heute aus betrachtet kann ich froh sein, dass mein Vater weder Pe-la-ge-ya
-Po-lu-ba-ri-nova-Ko-chi-na“ - ich spreche den Namen extra langsam aus und betone
-jede Silbe,„eine russische Göttin der Hydrodynamik, noch Lyud-mila Kel-dysh,
-Mathematikprofessorin an der Moskauer Lomonossow-Universität, kannte.“ Sini
-lacht auf: \„Ja, das wäre endlos schlimmer.“ „Absolut“ erwidere ich und lehne
-mich zurück, „am Ende haben mich sowieso alle nur Eddie genannt, sogar die
-Lehrer.“\ „Eddie?“ Sini lacht. „Das klingt ja viel cooler. Wie ein belgischer
-Fußballspieler oder ein britischer Reggae-Sänger.“ Sina springt von der Bank auf
-und tut so, als ob sie Fußballspielerin und Sängerin zugleich wäre. Ich falle
-vor Lachen beinahe von der Bank. Es ist so schön mit Sini Zeit zu verlieren. Der
-Wind weht mir den Geruch von Seealgen und Rauch in die Nase. Ich streiche über
-den Stoff meiner verwaschenen roten Trainingshose – ein Stück Dresden, das sich
-hier manchmal fremd anfühlt.
+irgendwie schräg.“\ „Ja, oder? Aber hätte auch schlimmer kommen können, weißt
+du? Von heute aus betrachtet kann ich froh sein, dass mein Vater weder
+Pe-la-ge-ya Po-lu-ba-ri-nova-Ko-chi-na“ – ich spreche den Namen extra langsam
+aus und betone jede Silbe, „eine russische Göttin der Hydrodynamik, noch
+Lyud-mila Kel-dysh, Mathematikprofessorin an der Moskauer
+Lomonossow-Universität, kannte.“ Sini lacht auf: „Ja, das wäre endlos
+schlimmer.“ „Absolut“, erwidere ich und lehne mich zurück, „am Ende haben mich
+sowieso alle nur Eddie genannt, sogar die Lehrer.“\ „Eddie?“ Sini lacht. „Das
+klingt ja viel cooler. Wie ein belgischer Fußballspieler oder ein britischer
+Reggae-Sänger.“ Sini springt von der Bank auf und tut so, als ob sie
+Fußballspielerin und Sängerin zugleich wäre. Ich falle vor Lachen beinahe von
+der Bank. Es ist so schön, mit Sini Zeit zu verlieren. Der Wind weht mir den
+Geruch von Seealgen und Rauch in die Nase. Ich streiche über den Stoff meiner
+verwaschenen roten Trainingshose – ein Stück Dresden, das sich hier manchmal
+fremd anfühlt.