--- old
+++ new
@@ -17,14 +17,1 @@
-  "feedback": {
-    "anni": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "lucia": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "soek": {
-      "rating": ".....",
-      "comments": ""
-    }
-  }
+  "proofreading": {}
@@ -59,2 +46,2 @@
-ausstößt. „Was ist Liebe?", flüstere ich. „Ist es Feuer? Oder ist es, wenn man
-jemanden ansieht und sich plötzlich nicht mehr so allein fühlt?"
+ausstößt. „Was ist Liebe?“, flüstere ich. „Ist es Feuer? Oder ist es, wenn man
+jemanden ansieht und sich plötzlich nicht mehr so allein fühlt?“
@@ -89,1 +76,1 @@
-„Was ist Liebe überhaupt?!" fauche ich leise in die Dunkelheit, so heftig, dass
+„Was ist Liebe überhaupt?!“ fauche ich leise in die Dunkelheit, so heftig, dass
@@ -92,1 +79,1 @@
-endlich nicht mehr so verloren fühlt?"
+endlich nicht mehr so verloren fühlt?“
@@ -95,1 +82,1 @@
-toll, du weißt es auch nicht."
+toll, du weißt es auch nicht.“