--- old
+++ new
@@ -72,4 +72,4 @@
-Küsse haben mich eher erschreckt, ich konnte nie dasselbe zurückgeben.
-Trotzdem... wenn ich an Liebe denke, dann ist sie das Erste, was mir einfällt.
-War das nun Liebe? Oder einfach nur, dass ich jemanden endlich mal wichtiger
-fand als mich selbst?
+Küsse haben mich eher erschreckt, ich konnte nie dasselbe zurückgeben. Trotzdem…
+wenn ich an Liebe denke, dann ist sie das Erste, was mir einfällt. War das nun
+Liebe? Oder einfach nur, dass ich jemanden endlich mal wichtiger fand als mich
+selbst?