--- old
+++ new
@@ -20,15 +20,2 @@
-  "feedback": {
-    "anni": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "lucia": {
-      "rating": ".....",
-      "comments": ""
-    },
-    "soek": {
-      "rating": ".....",
-      "comments": ""
-    }
-  },
-  "todo": "Redundanz zu 'Hof hinter dem Haus"
+  "todo": "Redundanz zu 'Hof hinter dem Haus",
+  "proofreading": {}
@@ -66,1 +53,1 @@
-Mädchen, das man bemitleiden muss." Nein. Ich dachte: „Da ist jemand Schönes."
+Mädchen, das man bemitleiden muss.“ Nein. Ich dachte: „Da ist jemand Schönes.“
@@ -104,1 +91,1 @@
-„*Da ist jemand Schönes.*"
+„*Da ist jemand Schönes.*“
@@ -110,1 +97,1 @@
-„*Dankbarkeit ist nicht Liebe.*" flüstere ich, und meine Stimme klingt fremd.
+„*Dankbarkeit ist nicht Liebe.*“ flüstere ich, und meine Stimme klingt fremd.
@@ -112,1 +99,1 @@
-„Hörst du das, Kleiner? Ich soll nicht dankbar sein. Ich soll… lieben können."
+„Hörst du das, Kleiner? Ich soll nicht dankbar sein. Ich soll… lieben können.“
@@ -118,1 +105,1 @@
-„*Ich verspreche, dass ich nie wegsehen werde.*"
+„*Ich verspreche, dass ich nie wegsehen werde.*“
@@ -126,1 +113,1 @@
-könnte ihm glauben." Dann falte ich den Brief, lege ihn direkt unter mein Herz,
+könnte ihm glauben.“ Dann falte ich den Brief, lege ihn direkt unter mein Herz,