--- old
+++ new
@@ -38,2 +38,1 @@
-Meine nächste Etappe führt mich nur wenig Kilometer durch den Wald. Ich will mich langsam an die großen Etappen 
-Ich stehe in dieser Hütte und kann kaum glauben, dass sie mir für eine Nacht
+Meine nächste Etappe führt mich nur wenig Kilometer durch den Wald. Ich will mich langsam an die großen Etappen heranwagen. Und wieder stehe ich vor einer dieser wunderbaren Hütten und kann kaum glauben, dass sie mir für eine Nacht