--- old
+++ new
@@ -65,1 +65,1 @@
-„Wir müssen reden“, beginnt sie. „So… wie es weitergehen soll.“
+„Wir müssen reden“, beginnt sie. „So … wie es weitergehen soll.“
@@ -73,1 +73,1 @@
-„Zurück…“ Meine Stimme bricht fast. Zurück, das heißt: in die Hände von Mielke.
+„Zurück …“ Meine Stimme bricht fast. Zurück, das heißt: in die Hände von Mielke.
@@ -102,2 +102,2 @@
-kostet zwar eine Menge Geld, ja. Aber mach dir keine Sorgen, Eddi. Ich übernehme
-das.“
+kostet zwar eine Menge Geld, ja. Aber mach dir keine Sorgen, Eddie. Ich
+übernehme das.“
@@ -115,2 +115,2 @@
-Und trotzdem… in mir wächst die Freude ihr mitzuteilen, dass ich das nicht schon
-wieder auf Raijas Schultern ablade muss.
+Und trotzdem … in mir wächst die Freude, ihr mitzuteilen, dass ich das nicht
+schon wieder auf Raijas Schultern abladen muss.