--- old
+++ new
@@ -42,1 +42,1 @@
-mich hier... als wäre ich nicht wirklich Teil davon."
+mich hier… als wäre ich nicht wirklich Teil davon."
@@ -48,1 +48,1 @@
-im Bus, auf der Straße... ich bin immer der, der anders aussieht."
+im Bus, auf der Straße… ich bin immer der, der anders aussieht."
@@ -58,1 +58,1 @@
-anschauen, als wärst du... von woanders."
+anschauen, als wärst du… von woanders."
@@ -62,1 +62,1 @@
-Jacke, den schmutzigen Schuhen. Aber jetzt... jetzt verstehe ich, dass er auch
+Jacke, den schmutzigen Schuhen. Aber jetzt… jetzt verstehe ich, dass er auch
@@ -124,1 +124,1 @@
-Neugier. Nicht nur Ablenkung. Es wäre... weil ich es will. Weil er es ist.
+Neugier. Nicht nur Ablenkung. Es wäre… weil ich es will. Weil er es ist.
@@ -144,1 +144,1 @@
-auf der Flucht. Nur diesmal ist es kein Grauen. Es ist... etwas anderes.
+auf der Flucht. Nur diesmal ist es kein Grauen. Es ist… etwas anderes.