--- old
+++ new
@@ -5,21 +5,30 @@
-    "date": "1985-07-04",
-    "daytime": "evening",
-    "place": "Joutsenlampi",
-    "persons": ["Eddie", "Aaron", "Demian", "kleiner deutscher Junge", "Schwedischer Junge #1",  "Schwedischer Junge #2","Mielke" ],
-    "synopsis": "Eddie gewinnt beim Pokerspiel gegen Aaron, tauscht das Geld in Markka um und trifft danach auf Mielke.",
-    "mood": "tense but triumphant",
-    "mode": "flashback",   // flashback | mainline
-    "illustrations": ["FIN_Jotsenlampi_Poker.png"],
-    "feedback": {
-        "anni": {
-            "rating":".....",
-            "comments":""
-        },
-        "lucia": {
-            "rating":".....",
-            "comments":""
-        },
-        "soek": {
-            "rating":".....",
-            "comments":""
-        }
+  "date": "1985-07-04",
+  "daytime": "evening",
+  "place": "Joutsenlampi",
+  "persons": [
+    "Eddie",
+    "Aaron",
+    "Demian",
+    "kleiner deutscher Junge",
+    "Schwedischer Junge #1",
+    "Schwedischer Junge #2",
+    "Mielke"
+  ],
+  "synopsis": "Eddie gewinnt beim Pokerspiel gegen Aaron, tauscht das Geld in Markka um und trifft danach auf Mielke.",
+  "mood": "tense but triumphant",
+  "mode": "flashback",
+  "illustrations": [
+    "FIN_Jotsenlampi_Poker.png"
+  ],
+  "feedback": {
+    "anni": {
+      "rating": ".....",
+      "comments": ""
+    },
+    "lucia": {
+      "rating": ".....",
+      "comments": ""
+    },
+    "soek": {
+      "rating": ".....",
+      "comments": ""
@@ -27,0 +36,1 @@
+  }
@@ -32,3 +42,4 @@
-Gelegenheit, meine paar D-Mark die ich hatte, in finnische Markka zu tauschen. In meinem
-blauen Kleidchen, dieser grässlichen Mischung aus Schleifen und Spangen und mit
-Edmillas mädchenhafter Attitüde war ich offenbar ein gern gesehener Gast.
+Gelegenheit, meine paar D-Mark die ich hatte, in finnische Markka zu tauschen.
+In meinem blauen Kleidchen, dieser grässlichen Mischung aus Schleifen und
+Spangen und mit Edmillas mädchenhafter Attitüde war ich offenbar ein gern
+gesehener Gast.
@@ -37,4 +48,4 @@
-wolle. Beides war gelogen, doch Edmilla verstand es ausgezeichnet, die naive Kleine zu
-spielen. Also erklärten sie mir großzügig ihre Strategien und Überlegungen.
-Auf Angeberei über mathematische Fähigkeiten brauchte man in dieser Runde nicht
-lange zu erwarten.
+wolle. Beides war gelogen, doch Edmilla verstand es ausgezeichnet, die naive
+Kleine zu spielen. Also erklärten sie mir großzügig ihre Strategien und
+Überlegungen. Auf Angeberei über mathematische Fähigkeiten brauchte man in
+dieser Runde nicht lange zu erwarten.
@@ -47,5 +58,5 @@
-Umgekehrt galt: Ein guter Mathematiker war noch lange kein
-guter Pokerspieler. Und genau das demonstrierten die Jungs ziemlich
-eindrucksvoll. Schon beim Aufnehmen der Karten konnte ich in ihren Augen lesen,
-was sie da gezogen hatten. Am Ende war ich so geübt, dass ich bei vier von
-fünf Spielen sofort erkennen konnte, wer die besten Karten hielt.
+Umgekehrt galt: Ein guter Mathematiker war noch lange kein guter Pokerspieler.
+Und genau das demonstrierten die Jungs ziemlich eindrucksvoll. Schon beim
+Aufnehmen der Karten konnte ich in ihren Augen lesen, was sie da gezogen hatten.
+Am Ende war ich so geübt, dass ich bei vier von fünf Spielen sofort erkennen
+konnte, wer die besten Karten hielt.
@@ -57,3 +68,3 @@
-blonder, groß gewachsener Junge – eigentlich hätte er mir gefallen können.
-Doch er war ein Riesenidiot. Auch die anderen aus der westdeutschen Delegation
-waren manchmal anstrengend, wenn auch eher unabsichtlich, wenn sie von ihren
+blonder, groß gewachsener Junge – eigentlich hätte er mir gefallen können. Doch
+er war ein Riesenidiot. Auch die anderen aus der westdeutschen Delegation waren
+manchmal anstrengend, wenn auch eher unabsichtlich, wenn sie von ihren
@@ -61,2 +72,2 @@
-„Ach, neulich, als ich in Rom war…“
-„Später werde ich auf jeden Fall mal in London arbeiten…“
+„Ach, neulich, als ich in Rom war…“ „Später werde ich auf jeden Fall mal in
+London arbeiten…“
@@ -70,8 +81,7 @@
-Ich bin mir sicher: den meisten war gar nicht bewusst, wie weh uns das tat.
-Aber Aaron wusste es. Ganz genau. Er setzte seine Angebereien gezielt bei
-Mädchen aus Osteuropa ein. Eine hübsche Ungarin hatte ihn schon abblitzen
-lassen, und dann war ich an der Reihe. Er roch wie ein wandelnder Intershop –
-diese Läden, in denen Westware gegen eine Pseudowährung verkauft wurde. Der
-typische Geruch nach Seife und Schokolade war jedem vertraut. Weniger davon
-wäre vielleicht sogar angenehm gewesen. Doch in seiner Dosis war es schlicht
-aufdringlich.
+Ich bin mir sicher: den meisten war gar nicht bewusst, wie weh uns das tat. Aber
+Aaron wusste es. Ganz genau. Er setzte seine Angebereien gezielt bei Mädchen aus
+Osteuropa ein. Eine hübsche Ungarin hatte ihn schon abblitzen lassen, und dann
+war ich an der Reihe. Er roch wie ein wandelnder Intershop – diese Läden, in
+denen Westware gegen eine Pseudowährung verkauft wurde. Der typische Geruch nach
+Seife und Schokolade war jedem vertraut. Weniger davon wäre vielleicht sogar
+angenehm gewesen. Doch in seiner Dosis war es schlicht aufdringlich.
@@ -88,4 +98,4 @@
-Und immer wieder forderte er mich auf mitzuspielen. Die anderen bestärkten
-ihn auch noch darin; sie hatten wohl Spaß daran, ein Mädchen verlieren zu sehen.
-Ich lehnte jedes Mal ab: „Ich kann das Spiel nicht und habe auch keine Währung,
-die euch interessiert.“
+Und immer wieder forderte er mich auf mitzuspielen. Die anderen bestärkten ihn
+auch noch darin; sie hatten wohl Spaß daran, ein Mädchen verlieren zu sehen. Ich
+lehnte jedes Mal ab: „Ich kann das Spiel nicht und habe auch keine Währung, die
+euch interessiert.“
@@ -95,7 +105,6 @@
-„Im Osten gar nicht, im Westen eins zu zehn“, erklärte ich.
-„Also 100 Ostmark sind 10 Westmark?“
-„Ja.“
-„Und wie viel verdient man so in der DDR?“
-„Mein Vater ist Ingenieur an der Uni. Er bekommt etwa 1000 netto.“
-„1000 netto? Das sind ja gerade mal 100 D-Mark!“ Aaron schnaubte. „Mein Bruder
-ist Anwalt, der verdient so viel in der Stunde.“ „Na toll!“ rief ich angewidert. Da war er wieder – Aaron, der Idiot.
+„Im Osten gar nicht, im Westen eins zu zehn“, erklärte ich. „Also 100 Ostmark
+sind 10 Westmark?“ „Ja.“ „Und wie viel verdient man so in der DDR?“ „Mein Vater
+ist Ingenieur an der Uni. Er bekommt etwa 1000 netto.“ „1000 netto? Das sind ja
+gerade mal 100 D-Mark!“ Aaron schnaubte. „Mein Bruder ist Anwalt, der verdient
+so viel in der Stunde.“ „Na toll!“ rief ich angewidert. Da war er wieder –
+Aaron, der Idiot.
@@ -103,1 +112,3 @@
-Immerhin begriff er jetzt, warum ich nicht mitspielte. „Das ist echt schade“, meinte einer der freundlicheren Jungs. „Kommt, wir legen alle was zusammen, damit Eddi mitspielen kann.“
+Immerhin begriff er jetzt, warum ich nicht mitspielte. „Das ist echt schade“,
+meinte einer der freundlicheren Jungs. „Kommt, wir legen alle was zusammen,
+damit Eddi mitspielen kann.“
@@ -105,1 +116,3 @@
-Ich zierte mich. „Ich kann doch kein Geld für nichts annehmen.“ „Wenn du gewinnst, gibst du’s zurück. Wenn du verlierst, haben wir es eh wieder.“ „Und am Ende kriegt sowieso alles Aaron“, murmelte ein anderer.
+Ich zierte mich. „Ich kann doch kein Geld für nichts annehmen.“ „Wenn du
+gewinnst, gibst du’s zurück. Wenn du verlierst, haben wir es eh wieder.“ „Und am
+Ende kriegt sowieso alles Aaron“, murmelte ein anderer.
@@ -107,1 +120,6 @@
-Na gut, dachte ich, und ließ mich überreden. Jeder gab mir fünf Mark. Die Jetons hatten sie aus irgendeinem Brettspiel geklaut: Würfel zählten fünf Mark, Spielfiguren eine. So bekam ich eine Handvoll Spielsteine, ohne das Gefühl, wirklich von den Jungs „bezahlt“ worden zu sein. 25 Westmark – das waren 250 Ostmark. So viel Geld hatte bekommen, als ich den Ferien vier Wochen lang im Uni-Archiv blödes Zeug katalogisiert habe. Und jetzt lag das in meiner Hand.
+Na gut, dachte ich, und ließ mich überreden. Jeder gab mir fünf Mark. Die Jetons
+hatten sie aus irgendeinem Brettspiel geklaut: Würfel zählten fünf Mark,
+Spielfiguren eine. So bekam ich eine Handvoll Spielsteine, ohne das Gefühl,
+wirklich von den Jungs „bezahlt“ worden zu sein. 25 Westmark – das waren 250
+Ostmark. So viel Geld hatte bekommen, als ich den Ferien vier Wochen lang im
+Uni-Archiv blödes Zeug katalogisiert habe. Und jetzt lag das in meiner Hand.
@@ -109,1 +127,6 @@
-Das machte mich vorsichtig. Ich spielte konservativ, mied Risiken, gewann selten – und verlor über den Abend hinweg etwa vier Mark. Die Jungs waren zufrieden. Ich gab keinem das Gefühl, besser zu sein und jeder dachte, er würde seinen Einsatz von mir doch irgendwann zurückbekommen. Währenddessen kippten die anderen Jungs ein Bier nach dem anderen, wurden albern, während ich bewusst nüchtern blieb.
+Das machte mich vorsichtig. Ich spielte konservativ, mied Risiken, gewann selten
+– und verlor über den Abend hinweg etwa vier Mark. Die Jungs waren zufrieden.
+Ich gab keinem das Gefühl, besser zu sein und jeder dachte, er würde seinen
+Einsatz von mir doch irgendwann zurückbekommen. Währenddessen kippten die
+anderen Jungs ein Bier nach dem anderen, wurden albern, während ich bewusst
+nüchtern blieb.
@@ -111,1 +134,7 @@
-Dann kam der Moment, auf den ich gewartet hatte. Aaron hob seine Karten auf, verzog kurz das Gesicht, setzte aber sofort seine überhebliche Miene auf und klopfte demonstrativ auf den Tisch. Ein Bluff. Ich hatte es schon die Tage zuvor bei ihm beobachtet. Zu meinen zwei Damen gesellte sich eine dritte. Ich erhöhte den Einsatz vorsichtig. Aaron zog mit. Einer nach dem anderen stieg aus, bis nur wir beide übrig waren. Fast mein ganzes „Vermögen“ lag schon im Pot, da ging Aaron plötzlich All-In.
+Dann kam der Moment, auf den ich gewartet hatte. Aaron hob seine Karten auf,
+verzog kurz das Gesicht, setzte aber sofort seine überhebliche Miene auf und
+klopfte demonstrativ auf den Tisch. Ein Bluff. Ich hatte es schon die Tage zuvor
+bei ihm beobachtet. Zu meinen zwei Damen gesellte sich eine dritte. Ich erhöhte
+den Einsatz vorsichtig. Aaron zog mit. Einer nach dem anderen stieg aus, bis nur
+wir beide übrig waren. Fast mein ganzes „Vermögen“ lag schon im Pot, da ging
+Aaron plötzlich All-In.
@@ -113,2 +142,3 @@
-„Was soll das?“ fragte Demian, der neben ihm saß. „Eddie hat höchstens noch fünf Mark.“
-„Und was sind die Regeln in so einem Fall?“ fragte ich. Ich wusste es wirklich nicht.
+„Was soll das?“ fragte Demian, der neben ihm saß. „Eddie hat höchstens noch fünf
+Mark.“ „Und was sind die Regeln in so einem Fall?“ fragte ich. Ich wusste es
+wirklich nicht.
@@ -117,4 +147,4 @@
-gehabt hatte, erklärte: „Dann gibt’s einen Nebenpot. Von Aarons All-In
-gehen nur fünf Mark in den Hauptpot, die anderen bleiben im Nebenpot.“
-„Also total sinnlos“, meinte ich. „Er kriegt das Geld im Grunde sowieso zurück.“
-„Stimmt schon“, bestätigte der Kleine.
+gehabt hatte, erklärte: „Dann gibt’s einen Nebenpot. Von Aarons All-In gehen nur
+fünf Mark in den Hauptpot, die anderen bleiben im Nebenpot.“ „Also total
+sinnlos“, meinte ich. „Er kriegt das Geld im Grunde sowieso zurück.“ „Stimmt
+schon“, bestätigte der Kleine.
@@ -123,2 +153,2 @@
-fing sich aber sofort. „Ich habe das beste Blatt des Tages!“
-„Vielleicht die besten Luschen“, entgegnete ich trocken. Alle lachten.
+fing sich aber sofort. „Ich habe das beste Blatt des Tages!“ „Vielleicht die
+besten Luschen“, entgegnete ich trocken. Alle lachten.
@@ -129,1 +159,2 @@
-Aufnehmen der Karten guckt er grimmig, und dann wechselt er auf diese aufgesetzte Fröhlichkeit.“
+Aufnehmen der Karten guckt er grimmig, und dann wechselt er auf diese
+aufgesetzte Fröhlichkeit.“
@@ -135,2 +166,3 @@
-„Moment! Nein, das bleibt liegen.“ forderte Demian, zu mir gewandt sagte er leise
-„Wir könnten dich unterstützen, Eddie, mir wäre es einiges wert, Aaron heute verlieren zu sehen.“
+„Moment! Nein, das bleibt liegen.“ forderte Demian, zu mir gewandt sagte er
+leise „Wir könnten dich unterstützen, Eddie, mir wäre es einiges wert, Aaron
+heute verlieren zu sehen.“
@@ -138,3 +170,3 @@
-Am Ende borgten sie mir tatsächlich genug Jetons, um mitzuhalten.
-Aaron bestand darauf: „Wenn Eddie verliert, schuldet ihr mir das Geld
-wirklich.“ – „Schon klar“, winkten die anderen ab.
+Am Ende borgten sie mir tatsächlich genug Jetons, um mitzuhalten. Aaron bestand
+darauf: „Wenn Eddie verliert, schuldet ihr mir das Geld wirklich.“ – „Schon
+klar“, winkten die anderen ab.
@@ -142,4 +174,4 @@
-Schließlich legte ich meine drei Damen offen auf den Tisch. Ein Raunen
-ging durch die Gruppe. Aaron fluchte, warf seine Karten verdeckt weg und rauschte ab.
-„Hey Aaron“, rief ich ihm hinterher, „du bekommst noch fünf Mark
-von mir!“ Er machte nur eine abfällige Geste.
+Schließlich legte ich meine drei Damen offen auf den Tisch. Ein Raunen ging
+durch die Gruppe. Aaron fluchte, warf seine Karten verdeckt weg und rauschte ab.
+„Hey Aaron“, rief ich ihm hinterher, „du bekommst noch fünf Mark von mir!“ Er
+machte nur eine abfällige Geste.
@@ -151,8 +183,6 @@
-Die Gruppe löste sich langsam auf. Ich rief Demian noch zu: „Danke! Und
-sag mal, weißt du, wie ich die D-Mark in finnische Markka tauschen kann?“
-„Wie viel willst du tauschen?“
-„Alles.“
-Er sah überrascht aus. „Das willst du hier alles ausgeben?“ Hm stimmt, sehr
-auffällig, dachte ich und Edmilla übernahm:
-„Ja… vielleicht ein Kleidchen, wie die finnischen Mädchen sie tragen. Die
-gefallen dir doch auch, oder?“ – Er errötete.
+Die Gruppe löste sich langsam auf. Ich rief Demian noch zu: „Danke! Und sag mal,
+weißt du, wie ich die D-Mark in finnische Markka tauschen kann?“ „Wie viel
+willst du tauschen?“ „Alles.“ Er sah überrascht aus. „Das willst du hier alles
+ausgeben?“ Hm stimmt, sehr auffällig, dachte ich und Edmilla übernahm: „Ja…
+vielleicht ein Kleidchen, wie die finnischen Mädchen sie tragen. Die gefallen
+dir doch auch, oder?“ – Er errötete.
@@ -160,3 +190,3 @@
-Kurz darauf kam er mit einem Bündel Scheine zurück. „Im Kästchen waren 133
-Mark. Der Kurs lag zuletzt bei eins zu zwei-komma-zwei. Das macht freundlich aufgerundet 300
-Markka.“ Er drückte sie mir in die Hand.
+Kurz darauf kam er mit einem Bündel Scheine zurück. „Im Kästchen waren 133 Mark.
+Der Kurs lag zuletzt bei eins zu zwei-komma-zwei. Das macht freundlich
+aufgerundet 300 Markka.“ Er drückte sie mir in die Hand.
@@ -164,2 +194,2 @@
-Ich tat, was Edmilla getan hätte, und gab ihm einen Kuss auf die Wange.
-„Das verdanke ich dir.“
+Ich tat, was Edmilla getan hätte, und gab ihm einen Kuss auf die Wange. „Das
+verdanke ich dir.“
@@ -168,2 +198,2 @@
-wirklich oft schäbig. Außerdem war es ein Vergnügen, Aaron so hart
-verlieren zu sehen.“
+wirklich oft schäbig. Außerdem war es ein Vergnügen, Aaron so hart verlieren zu
+sehen.“
@@ -177,1 +207,5 @@
-Raija hatte die ganze Zeit geduldig zugehört. Sie musterte mich mit einem dieser schiefen Lächeln, bei denen ich nie wusste, ob sie mich tadeln oder in den Arm nehmen wollte.Raija hatte die ganze Zeit geduldig zugehört. Sie mustert mich mit einem dieser schiefen Lächeln, bei denen ich nie weiß, ob sie mich tadeln oder in den Arm nehmen will.
+Raija hatte die ganze Zeit geduldig zugehört. Sie musterte mich mit einem dieser
+schiefen Lächeln, bei denen ich nie wusste, ob sie mich tadeln oder in den Arm
+nehmen wollte.Raija hatte die ganze Zeit geduldig zugehört. Sie mustert mich mit
+einem dieser schiefen Lächeln, bei denen ich nie weiß, ob sie mich tadeln oder
+in den Arm nehmen will.
@@ -179,1 +213,2 @@
-„Du hast Nerven, Mädchen“, sagt sie leise. „So ein Spiel gegen Aaron – und dann auch noch gewonnen.“
+„Du hast Nerven, Mädchen“, sagt sie leise. „So ein Spiel gegen Aaron – und dann
+auch noch gewonnen.“
@@ -183,1 +218,3 @@
-„Dreihundert Markka sind viel Geld, wenn man sie mit nach Hause nimmt. In der DDR könnte man davon vermutliche einige Monate leben. Aber hier in Finnland reicht es kaum für eine Zugfahrkarte und ein paar warme Mahlzeiten.“
+„Dreihundert Markka sind viel Geld, wenn man sie mit nach Hause nimmt. In der
+DDR könnte man davon vermutliche einige Monate leben. Aber hier in Finnland
+reicht es kaum für eine Zugfahrkarte und ein paar warme Mahlzeiten.“
@@ -185,1 +222,3 @@
-Ihre Stimme bekommt plötzlich etwas Ernstes, fast Mütterliches. „Versteh mich nicht falsch, ich bin beeindruckt. Dein Geschick, dein Mut – Respekt. Aber verlass dich nicht nur auf dein Glück. Es trägt dich nicht bis ans Ziel.“
+Ihre Stimme bekommt plötzlich etwas Ernstes, fast Mütterliches. „Versteh mich
+nicht falsch, ich bin beeindruckt. Dein Geschick, dein Mut – Respekt. Aber
+verlass dich nicht nur auf dein Glück. Es trägt dich nicht bis ans Ziel.“
@@ -187,1 +226,2 @@
-Ich nicke, auch wenn es mir in den Fingern juckt zu widersprechen. Sie hat ja recht, leider.
+Ich nicke, auch wenn es mir in den Fingern juckt zu widersprechen. Sie hat ja
+recht, leider.
@@ -191,1 +231,3 @@
-Für einen Moment legt sie mir die Hand auf die Schulter, fest, beinahe beschützend. Dann lacht sie, als wolle sie die Schwere vertreiben. „Und jetzt ab ins Bett, Pokerspielerin. Morgen wird ein langer Tag.“
+Für einen Moment legt sie mir die Hand auf die Schulter, fest, beinahe
+beschützend. Dann lacht sie, als wolle sie die Schwere vertreiben. „Und jetzt ab
+ins Bett, Pokerspielerin. Morgen wird ein langer Tag.“
@@ -193,1 +235,2 @@
-Wir wünschen uns gute Nacht. Zwei Welten in mir: die Schwere ihrer Warnung – und die Leichtigkeit meines kleinen Sieges.
+Wir wünschen uns gute Nacht. Zwei Welten in mir: die Schwere ihrer Warnung – und
+die Leichtigkeit meines kleinen Sieges.
@@ -195,1 +238,2 @@
-Damit gehe ich schlafen. Dreihundert Markka sind kein Fallschirm, aber sie sind ein Griff.
+Damit gehe ich schlafen. Dreihundert Markka sind kein Fallschirm, aber sie sind
+ein Griff.