--- old
+++ new
@@ -65,2 +65,2 @@
-> **➡ Mit Wahrscheinlichkeitslehre zum Pokergenie...**\
-> (...dafür bräuchte Eddie ein ganzes Buch, aber ein paar einleuchtende Tricks für Poker findest Du im Kapitel 3 von „Eddie rechnet“.)
+> **➡ Mit Wahrscheinlichkeitslehre zum Pokergenie…**\
+> (…dafür bräuchte Eddie ein ganzes Buch, aber ein paar einleuchtende Tricks für Poker findest Du im Kapitel 3 von „Eddie rechnet“.)