--- old
+++ new
@@ -3,1 +3,1 @@
-# Nachspiel / Nachtwache
+# Schützengräben
@@ -20,0 +20,1 @@
+```
@@ -21,0 +22,1 @@
+Ich wache wieder auf. Die Wohnung ist klein und mollig gemütlich. Es riecht nach Tee und Seife. Gardinen filtern das Licht weich, ein Poster hängt schief, daneben eine gestrickte bunte Borte, vermutlich irgendeine finnische Volklore. Alles ist einfach, aber warm.
@@ -22,4 +24,0 @@
----
-
-Ich wache wieder auf. Die Wohnung ist klein und mollig gemütlich. Es riecht nach Tee und Seife, nach Stoffen, die Sonne gesehen haben, und nach einer Küche, die mehr Kaffee kennt als Essen. Gardinen filtern das Licht weich, ein Poster hängt schief, daneben eine gestrickte Sami-Borte. Alles ist einfach, aber warm.
-