--- old
+++ new
@@ -89,1 +89,1 @@
-„Frei?“, stoße ich bitter hervor. „Du kriegst alles serviert – ja, sogar
+„Frei?“ stoße ich bitter hervor. „Du kriegst alles serviert – ja, sogar
@@ -103,1 +103,1 @@
-Also sitzen wir weiter im Hof, die Morgensonne weicht einem krellen Vormittag.
+Also sitzen wir weiter im Hof, die Morgensonne weicht einem grellen Vormittag.
@@ -105,1 +105,1 @@
-„Im Westen gibt’s Arbeitslosigkeit, Armut, Ausbeutung,“ hebt er an.
+„Im Westen gibt’s Arbeitslosigkeit, Armut, Ausbeutung“, hebt er an.
@@ -139,1 +139,1 @@
-„Jorma,“ sage ich leise, „ich brauche keine zehn Jahre. Ich hab es gelebt. Jeden
+„Jorma“, sage ich leise, „ich brauche keine zehn Jahre. Ich hab es gelebt. Jeden
@@ -164,2 +164,2 @@
-Stimmen – gedämpft, aber eindeutig Streit. Ich verstehe kein Wort, das Finnische
-klingt wie ein ständiges Hin und Her, doch die Stimmung ist klar.
+Stimmen – gedämpft, aber eindeutig ein Streit. Ich verstehe kein Wort, das
+Finnische klingt wie ein ständiges Hin und Her, doch die Stimmung ist klar.
@@ -167,1 +167,1 @@
-Jetzt habe ich doch Stress zwischen die beiden gebracht. Das ärgert mich.
+Jetzt habe ich doch Stress zwischen den beiden gebracht. Das ärgert mich.