--- old
+++ new
@@ -6,0 +6,1 @@
+  "day": "Friday",
@@ -16,3 +17,1 @@
-  "todo": [
-    "In der Stadt laufen Vorbereitungen für das große Festival, was in etwa einer Woche starten wird. Eddie überlegt kurz, ob einer der Fahrer sie mitnehmen könnte, verwirft die Ideen aber schließlich - Nein, Festival ist abgewählt, die Augenzeugen können sich nicht daran erinnern. Es stört den Fluss."
-  ],
+  "todo": "Maria weniger souverän zeichnen, auch ihr Mut muss erst wachsen",
@@ -72,2 +71,3 @@
-Wände, ein einfacher Turm. Keine Gläubigen. Kein Summen von Stimmen. Vielleicht
-sind alle am See, bei einem Fest.
+Wände, ein einfacher Turm. Keine Gläubigen, eigentlich keine Überraschung an
+einen Freitagnachmittag. Kein Summen von Stimmen. Wieder keine Menschenmenge in
+der ich verschwinden kann.
@@ -88,1 +88,1 @@
-Sie guckt mich an, unbewegt. Kein Stirnrunzeln, kein Kopfschütteln. Nur dieser
+Sie schaut mich an, unbewegt. Kein Stirnrunzeln, kein Kopfschütteln. Nur dieser