--- old
+++ new
@@ -80,1 +80,1 @@
-Kirchen gemacht. Kein schwerer Stein, keine goldenen Verzierungen wie in den
+Kirche gemacht. Kein schwerer Stein, keine goldenen Verzierungen wie in den
@@ -82,1 +82,1 @@
-durch jede Wand hindurchscheinen.Und da vorne – eine Frau. Steht ruhig, schaut
+durch jede Wand hindurchscheinen. Und da vorne – eine Frau. Steht ruhig, schaut
@@ -88,1 +88,1 @@
-Sie guckt mich an, unbewegt. Keine Stirnrunzel, kein Kopfschütteln. Nur dieser
+Sie guckt mich an, unbewegt. Kein Stirnrunzeln, kein Kopfschütteln. Nur dieser
@@ -97,1 +97,1 @@
-„Вы… вы говорите по-русски?“ Sie sprechen russisch, stottere ich, Worte stolpern
+„Вы… вы говорите по-русски?“ Sie sprechen Russisch, stottere ich, Worte stolpern