--- old
+++ new
@@ -94,4 +94,4 @@
-hinein, ein Ei platzt, das Eigelb schwappt gelb wie die Sonne, die gleich
-aufgehen wird. Toast springt, ich stelle Teller auf, und im Ofen backen Muffins,
-deren Duft die kleine Wohnung füllt. Alles wirkt festlicher, nur weil ich es für
-sie tue.
+hinein. Ein Ei platzt, und das Eigelb schwappt gelb wie die Sonne, die gleich
+aufgehen wird. Der Toast springt. Ich stelle Teller auf, und im Ofen backen
+Muffins, deren Duft die kleine Wohnung füllt. Alles wirkt festlicher, nur weil
+ich es für sie tue.
@@ -109,2 +109,2 @@
-Ich drehe mich um, halte das Bratfett tropfende Pfannenwender wie eine Trophäe
-hoch. „Studentenausweis.“
+Ich drehe mich um, halte den vom Bratfett tropfenden Pfannenwender wie eine
+Trophäe hoch. „Studentenausweis.“
@@ -126,1 +126,1 @@
-Buntstift übermalt. Zwei Doppelhelix, eine mit blauem Iro, die andere mit
+Buntstift übermalt. Zwei Doppelhelixen, eine mit blauem Iro, die andere mit
@@ -147,2 +147,2 @@
-Kruste bekommt. Sie tippt mit der Nase an mein Ohr. „Und die Muffins? Verbrennen
-die nicht?“
+Kruste bekommt. Sie tippt mit der Nase gegen mein Ohr. „Und die Muffins?
+Verbrennen die nicht?“
@@ -154,1 +154,1 @@
-ich meine Socken aus und strecke ihr die Füße entgegen. „Schau: alles verheilt.
+ich meine Socken aus und strecke ihr die Füße entgegen. „Schau: Alles verheilt.
@@ -157,2 +157,2 @@
-Sini nickt wohlwollend. „Sehr gern mein Hinkebein, ich habe auch eine
-Überaschung für dich, aber da musst du dich noch etwas gedulden.“
+Sini nickt wohlwollend. „Sehr gern, mein Hinkebein, ich habe auch eine
+Überraschung für dich, aber da musst du dich noch etwas gedulden.“