--- old
+++ new
@@ -50,19 +50,19 @@
-Ein Freund meines Vaters war begeisterter Ski-Langläufer. Er überredete meinen
-Vater, mit ihm die „Hohe Tour“ zu machen. Eine Ski-Langstreckenroute, die sich
-entlang der Grenze der DDR zur Tschechoslowakei erstreckte. Mit 50 km Länge und
-vielen steilen Abfahrten war sie technisch ausgesprochen anspruchsvoll. Mein
-Vater war wenig begeistert von der Idee, trainierte aber heimlich für dieses
-Ereignis. Wobei es da nicht um die Zeit oder die Platzierung ging - es war
-überhaupt kein offizielles Rennen - sondern es ging allein um den Umstand, sie
-überhaupt zu Ende zu bringen. Am Ende schaffte mein Vater die Tour, trotz
-mehrerer schwerer Stürze und Materialversagens seiner Ausrüstung. Danach war er
-eine Woche krank, so sehr hatte er sich verausgabt und seine Füße sahen aus wie
-eine einzige dicke Blase. Als sein Freund kurz danach wieder mal bei uns zu
-Besuch war, feierten sie ihre Tour feucht-fröhlich. Ich kann mich noch an einen
-Satz seines Freundes erinnern: „Das war unsere Vasaloppet!“ „Was ist die
-Vasaloppet?\“ habe ich als kleines Kind gefragt. Und dann erzählte er mir die
-Legende, von König Vasa in blumigen spannenden Worten: Es war ein bitterkalter
-Winter irgendwann Anfang des 16 Jahrhunderts. Der junge Prinz Gustav Vasa, ein
-Mann mit mutigem Herzen und festem Willen, war auf der Flucht. Hinter ihm jagten
-die Truppen des grausamen dänischen Königs her -- ein König, der das Land mit
-eiserner Hand beherrschte und seine Gegner gnadenlos verfolgte.
+Ein Freund meines Vaters war begeisterter Skilangläufer. Er überredete meinen
+Vater, mit ihm die „Hohe Tour“ zu machen. Eine Skilangstreckenroute, die sich
+entlang der Grenze der DDR zur Tschechoslowakei erstreckte. Mit 50 Kilometern
+Länge und vielen steilen Abfahrten war sie technisch ausgesprochen
+anspruchsvoll. Mein Vater war wenig begeistert von der Idee, trainierte aber
+heimlich für dieses Ereignis. Wobei es da nicht um die Zeit oder die Platzierung
+ging – es war überhaupt kein offizielles Rennen – sondern es ging allein um den
+Umstand, sie überhaupt zu Ende zu bringen. Am Ende schaffte mein Vater die Tour,
+trotz mehrerer schwerer Stürze und des Materialversagens seiner Ausrüstung.
+Danach war er eine Woche krank, so sehr hatte er sich verausgabt, und seine Füße
+sahen aus wie eine einzige dicke Blase. Als sein Freund kurz danach wieder mal
+bei uns zu Besuch war, feierten sie ihre Tour feucht-fröhlich. Ich kann mich
+noch an einen Satz seines Freundes erinnern: „Das war unsere Vasaloppet!“ „Was
+ist die Vasaloppet?“ habe ich als kleines Kind gefragt. Und dann erzählte er mir
+die Legende von König Vasa in blumigen, spannenden Worten: Es war ein
+bitterkalter Winter, irgendwann Anfang des 16. Jahrhunderts. Der junge Prinz
+Gustav Vasa, ein Mann mit mutigem Herzen und festem Willen, war auf der Flucht.
+Hinter ihm jagten die Truppen des grausamen dänischen Königs her – ein König,
+der das Land mit eiserner Hand beherrschte und seine Gegner gnadenlos verfolgte.
@@ -73,5 +73,5 @@
-erschöpft schnallte erdie Skier unter seine Füße und machte sich auf den Weg.
-Sein Ziel war die Grenze nach Norwegen -- dort hoffte er auf Schutz, weit weg
-von den dänischen Soldaten. Der Wind biss in sein Gesicht und die endlosen
-Wälder schienen ihn verschlingen zu wollen. Doch Gustav war schneller als die
-Dunkelheit, die sich um ihn legte.\ Unbemerkt von ihm hatten zwei seiner
+erschöpft schnallte er die Skier unter seine Füße und machte sich auf den Weg.
+Sein Ziel war die Grenze nach Norwegen – dort hoffte er auf Schutz, weit weg von
+den dänischen Soldaten. Der Wind biss in sein Gesicht und die endlosen Wälder
+schienen ihn verschlingen zu wollen. Doch Gustav war schneller als die
+Dunkelheit, die sich um ihn legte. Unbemerkt von ihm hatten zwei seiner
@@ -91,1 +91,1 @@
-Route, die Gustav damals nahm. Sie nennen es den Vasaloppet -- den Lauf des
+Route, die Gustav damals nahm. Sie nennen es den Vasaloppet – den Lauf des
@@ -94,1 +94,1 @@
-OK, der Freund meines Vaters konnte echt gut Geschichten erzählen. Ein
+Okay, der Freund meines Vaters konnte echt gut Geschichten erzählen. Ein
@@ -96,1 +96,1 @@
-das, was die beiden Männer mit der Hohen Tour verband? Fühlte sie sich so wie
+das, was die beiden Männer mit der Hohen Tour verband? Fühlten sie sich so wie
@@ -110,1 +110,1 @@
-ein viel erfreulicher Gedanke: ich bin auf meiner ganz persönlichen Vasaloppet.
+ein viel erfreulicher Gedanke: Ich bin auf meiner ganz persönlichen Vasaloppet.
@@ -112,4 +112,4 @@
-auch nicht 90km, sondern 400km Wanderweg. Mein Pilgerpfad in die Freiheit. Mir
-gefällt der Gedanke sehr und ich stelle mir meinen Vater und seinen Kumpel vor,
-wenn sie wüssten, dass ich meine eigene Vasaloppet - na was wohl? - natürlich zu
-Ende bringe.
+auch nicht 90 Kilometer, sondern 400 Kilometer Wanderweg. Mein Pilgerpfad in die
+Freiheit. Mir gefällt der Gedanke sehr und ich stelle mir meinen Vater und
+seinen Kumpel vor, wenn sie wüssten, dass ich meine eigene Vasaloppet – na, was
+wohl? – natürlich zu Ende bringe.
@@ -118,2 +118,2 @@
-Dresden, kein Mielke im Rücken. Nur Wälder, Sümpfe, Seen und ich -- mit meinem
-alten Armeeschlafsack und meiner schwarzen DDR Wanderstiefeln. Ich laufe, bis
+Dresden, kein Mielke im Rücken. Nur Wälder, Sümpfe, Seen und ich – mit meinem
+alten Armeeschlafsack und meinen schwarzen DDR-Wanderstiefeln. Ich laufe, bis